<!DOCTYPE html>
        <html>
        <head>
            <meta charset="UTF-8">
            <title>paper reading list</title>
            <style>
/* From extension vscode.github */
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

.vscode-dark img[src$=\#gh-light-mode-only],
.vscode-light img[src$=\#gh-dark-mode-only],
.vscode-high-contrast:not(.vscode-high-contrast-light) img[src$=\#gh-light-mode-only],
.vscode-high-contrast-light img[src$=\#gh-dark-mode-only] {
	display: none;
}

</style>
            
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">
<style>
            body {
                font-family: -apple-system, BlinkMacSystemFont, 'Segoe WPC', 'Segoe UI', system-ui, 'Ubuntu', 'Droid Sans', sans-serif;
                font-size: 14px;
                line-height: 1.6;
            }
        </style>
        <style>
.task-list-item {
    list-style-type: none;
}

.task-list-item-checkbox {
    margin-left: -20px;
    vertical-align: middle;
    pointer-events: none;
}
</style>
<style>
:root {
  --color-note: #0969da;
  --color-tip: #1a7f37;
  --color-warning: #9a6700;
  --color-severe: #bc4c00;
  --color-caution: #d1242f;
  --color-important: #8250df;
}

</style>
<style>
@media (prefers-color-scheme: dark) {
  :root {
    --color-note: #2f81f7;
    --color-tip: #3fb950;
    --color-warning: #d29922;
    --color-severe: #db6d28;
    --color-caution: #f85149;
    --color-important: #a371f7;
  }
}

</style>
<style>
.markdown-alert {
  padding: 0.5rem 1rem;
  margin-bottom: 16px;
  color: inherit;
  border-left: .25em solid #888;
}

.markdown-alert>:first-child {
  margin-top: 0
}

.markdown-alert>:last-child {
  margin-bottom: 0
}

.markdown-alert .markdown-alert-title {
  display: flex;
  font-weight: 500;
  align-items: center;
  line-height: 1
}

.markdown-alert .markdown-alert-title .octicon {
  margin-right: 0.5rem;
  display: inline-block;
  overflow: visible !important;
  vertical-align: text-bottom;
  fill: currentColor;
}

.markdown-alert.markdown-alert-note {
  border-left-color: var(--color-note);
}

.markdown-alert.markdown-alert-note .markdown-alert-title {
  color: var(--color-note);
}

.markdown-alert.markdown-alert-important {
  border-left-color: var(--color-important);
}

.markdown-alert.markdown-alert-important .markdown-alert-title {
  color: var(--color-important);
}

.markdown-alert.markdown-alert-warning {
  border-left-color: var(--color-warning);
}

.markdown-alert.markdown-alert-warning .markdown-alert-title {
  color: var(--color-warning);
}

.markdown-alert.markdown-alert-tip {
  border-left-color: var(--color-tip);
}

.markdown-alert.markdown-alert-tip .markdown-alert-title {
  color: var(--color-tip);
}

.markdown-alert.markdown-alert-caution {
  border-left-color: var(--color-caution);
}

.markdown-alert.markdown-alert-caution .markdown-alert-title {
  color: var(--color-caution);
}

</style>
        
        </head>
        <body class="vscode-body vscode-light">
            <h1 id="paper-reading-list">paper reading list</h1>
<!--  test
## lab and people 
 
[IBL](https://www.internationalbrainlab.com/our-team)

[Liam Paninski](https://scholar.google.com/citations?user=FayXlTYAAAAJ&hl=en)

[Jonathan Pillow](https://scholar.google.com/citations?user=-ElvJ9wAAAAJ&hl=en) -->
<h2 id="connectome">Connectome</h2>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-07558-y">Neuronal wiring diagram of an adult brain</a></p>
</li>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.add9330">The connectome of an insect brain</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1912.08986">Deep Connectomics Networks: Neural Network Architectures Inspired by Neuronal Networks</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.06.08.544257v1">Not everything, not everywhere, not all at once: a study of brain-wide encoding of movement</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.05.30.542725v1">Synaptic architecture of leg and wing motor control networks in Drosophila</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-07600-z">Synaptic architecture of leg and wing premotor control networks in Drosophila</a>  John C. Tuthill</p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/62362">Accelerating with FlyBrainLab the discovery of the functional logic of the Drosophila brain in the connectomic and synaptomic era</a></p>
</li>
<li>
<p><a href="https://elifesciences.org/reviewed-preprints/91532#tab-content">Deciphering the Genetic Code of Neuronal Type Connectivity: A Bilinear Modeling Approach</a>
bilinear + Mu Qiao</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41583-023-00756-z">How deep is the brain? The shallow brain hypothesis</a></p>
</li>
<li>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(23)00719-5">Mapping of neuronal and glial primary cilia contactome and connectome in the human cerebral cortex</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.04.16.589741v2">Connectome-driven neural inventory of a complete visual system</a>
Janelia
a new connectome of the right optic lobe from a male Drosophila central nervous system FIB-SEM volume</p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.10.04.616605v2">Generating brain-wide connectome using synthetic axonal morphologies</a>
Blue Brain Project</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41593-024-01812-2">The cell-type underpinnings of the human functional cortical connectome</a></p>
</li>
</ul>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.02.10.528036v3">Evolution of connectivity architecture in the Drosophila mushroom body</a></p>
<p><a href="https://elifesciences.org/articles/62576">The connectome of the adult Drosophila mushroom body provides insights into function</a></p>
<h3 id="technology-of-connectome">technology of connectome</h3>
<p><a href="https://openreview.net/forum?id=lHDVYDy5S9l">Bridging the Gap: Point Clouds for Merging Neurons in Connectomics</a>
MIDL 2022</p>
<p><a href="https://www.nature.com/articles/s41592-021-01183-7">Automatic detection of synaptic partners in a whole-brain Drosophila electron microscopy data set</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.11.04.467197v1.abstract">3D reconstruction of cell nuclei in a full Drosophila brain</a></p>
<h4 id="connection-reconsturction">connection reconsturction</h4>
<p><a href="https://arxiv.org/abs/1210.1544">Reconstruction of Sparse Circuits Using Multi-neuronal Excitation (RESCUME)</a></p>
<h3 id="connectome-analysis">connectome analysis</h3>
<p><a href="https://www.nature.com/articles/s41586-024-07968-y">Network statistics of the whole-brain connectome of Drosophila</a>
<strong>Mala Murthy, Nature</strong></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.06.27.546055v2">Whole-brain annotation and multi-connectome cell typing quantifies circuit stereotypy in Drosophila</a>
<strong>hemibrain and flywire</strong></p>
<p><a href="https://elifesciences.org/articles/57443">A connectome and analysis of the adult Drosophila central brain</a><br>
<strong>hemibrain</strong></p>
<p><a href="https://elifesciences.org/articles/79139">Connectomic analysis of the Drosophila lateral neuron clock cells reveals the synaptic basis of functional pacemaker classes</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1053811920308326">Cross-species functional alignment reveals evolutionary hierarchy within the connectome</a></p>
<p><a href="https://elifesciences.org/articles/66039">A connectome of the Drosophila central complex reveals network motifs suitable for flexible navigation and context-dependent action selection</a></p>
<p><a href="https://www.eneuro.org/content/9/2/ENEURO.0053-22.2022">Identifying Inputs to Visual Projection Neurons in Drosophila Lobula by Analyzing Connectomic Data</a></p>
<p><a href="https://www.nature.com/articles/s41593-024-01640-4">Hue selectivity from recurrent circuitry in Drosophila</a></p>
<p><a href="https://www.nature.com/articles/nn.2489">Diversity and wiring variability of olfactory local interneurons in the Drosophila antennal lobe</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0925231220317926">Multilayer network analysis of <strong>C. elegans</strong>: Looking into the locomotory circuitry</a>
<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005283">The Multilayer Connectome of Caenorhabditis <strong>elegans</strong></a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.11.30.569343v1.abstract">Exploring the Brain Characteristics of Structure-informed Functional Connectivity through Graph Attention Network</a></p>
<h3 id="connectome-and-gene">connectome and gene</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867421007583">Molecular topography of an entire nervous system</a>
<strong>key_and_lock</strong>
<a href="https://elifesciences.org/articles/64903#metrics">The Prop1-like homeobox gene unc-42 specifies the identity of synaptically connected neurons</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0960982223010631">Brain wiring determinants uncovered by integrating connectomes and transcriptomes</a>
<strong>Yoo2023</strong> T4_T5</p>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(20)30774-1">Transcriptional Programs of Circuit Assembly in the Drosophila Visual System</a> <strong>S. Lawrence Zipursky</strong></p>
<p><strong>bilinear</strong> <a href="https://www.pnas.org/doi/full/10.1073/pnas.2009093117">Uncovering the genetic blueprint of the C. elegans nervous system</a>
István A. Kovács, Dániel L. <strong>Barabási</strong>, and Albert-László <strong>Barabási</strong>  PNAS</p>
<p><strong>biclique motifs</strong> <a href="https://www.sciencedirect.com/science/article/pii/S0896627319309262">A Genetic Model of the Connectome</a>
Dániel L. Barabási 1, Albert-László Barabási</p>
<p><strong>bilinear</strong> <a href="https://elifesciences.org/reviewed-preprints/91532">Deciphering the Genetic Code of Neuronal Type Connectivity: A Bilinear Modeling Approach</a>
<strong>Mu Qiao</strong></p>
<p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010382"><strong>Constructive connectomics</strong>: How neuronal axons get from here to there using gene- expression maps derived from their family trees</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627319300091">Cell-type-Specific Patterned Stimulus-Independent Neuronal Activity in the Drosophila Visual System during Synapse Formation</a>
<strong>S. Lawrence Zipursky</strong></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867416300034">Bayesian Sparse Regression Analysis Documents the Diversity of Spinal Inhibitory Interneurons</a>
<strong>Bayesian method</strong></p>
<p><a href="https://elifesciences.org/articles/19942">Genome-wide identification of neuronal activity-regulated genes in Drosophila</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.08.07.606541v1">Stochastic Wiring of Cell Types Enhances Fitness by Generating Phenotypic Variability</a> Anthony Zador<br>
connection probabilities between discrete cell types are genetically specified, to investigate
the benefits of stochasticity in the development of neural wiring</p>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.12.14.628490v1">Dissecting origins of wiring specificity in dense cortical connectomes</a></p>
<p><a href="https://www.nature.com/articles/s41380-022-01519-5">Connectome gradient dysfunction in major depression and its association with gene expression profiles and treatment outcomes</a></p>
<h4 id="experiment-support">experiment support</h4>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1534580724001485?via%3Dihub">Spatial patterning controls neuron numbers in the Drosophila visual system</a></p>
<p><a href="https://www.nature.com/articles/s41586-024-07221-6">Whole-cortex in situ sequencing reveals input-dependent area identity</a>
Zador lab</p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627319300091">Cell-type-Specific Patterned Stimulus-Independent Neuronal Activity in the Drosophila Visual System during Synapse Formation</a></p>
<h3 id="cell-cell-communication-and-interaction">cell-cell communication and interaction</h3>
<p><a href="https://www.nature.com/articles/s41596-020-0292-x">CellPhoneDB: inferring <strong>cell–cell communication</strong> from combined expression of multi-subunit ligand–receptor complexes</a></p>
<p><a href="https://www.nature.com/articles/s41576-020-00292-x">Deciphering cell–cell interactions and communication from gene expression</a></p>
<p><a href="https://www.nature.com/articles/s41467-024-51329-2">Deciphering cell–cell communication at single-cell resolution for spatial transcriptomics with subgraph-based graph attention network</a></p>
<h3 id="protein-interaction">protein interaction</h3>
<p><a href="https://paperswithcode.com/dataset/ppi">PPI (Protein-Protein Interactions (PPI))</a></p>
<h4 id="experiment-support-1">experiment support</h4>
<p><a href="https://www.nature.com/articles/s41467-020-15981-8">DIP/Dpr interactions and the evolutionary design of specificity in protein families</a>
binding specificity of DIP/Dpr subgroups is controlled by “negative constraints”, which interfere with binding</p>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.07.08.602367v2">Nematode Extracellular Protein Interactome Expands Connections between Signaling Pathways</a></p>
<p><a href="https://www.nature.com/articles/s41597-024-03233-z">A comprehensive dataset of protein-protein interactions and ligand binding pockets for advancing drug discovery</a>
<strong>PPIs</strong></p>
<p><a href="https://arxiv.org/abs/1806.04398">Attentive cross-modal paratope prediction</a>
Antibodies</p>
<h4 id="deep-learning-methods">deep learning methods</h4>
<p>paper with code: <a href="https://paperswithcode.com/dataset/ppi">https://paperswithcode.com/dataset/ppi</a></p>
<p><a href="https://www.nature.com/articles/d41586-020-00018-3">Powerful antibiotics discovered using AI</a></p>
<p><a href="https://link.springer.com/article/10.1186/s12859-022-04910-9">Struct2Graph: a graph attention network for structure based predictions of protein–protein interactions</a></p>
<p><a href="https://academic.oup.com/bib/article/23/1/bbab502/6457163">Accurate protein function prediction via graph attention networks with predicted structure information</a></p>
<h4 id="result-verification">result verification</h4>
<p><a href="https://www.cell.com/cell/fulltext/S0092-8674(24)01093-6">A conserved fertilization complex bridges sperm and egg in vertebrates</a> <strong>AlphaFold-Multimer</strong>
This approach employs structural predictions of pairwise protein-protein interactions between each bait protein and the candidate proteins. A high interface-predicted template modeling score (<strong>ipTM score</strong>) reflects high confidence in the relative positioning of the interfaces.</p>
<p><img src="file:///d:\GitHub\lengyuner.github.io\blog\paper reading list\image.png" alt="alt text|50%"></p>
<h3 id="data-mapping-between-transcriptome-and-connectome">data mapping between transcriptome and connectome</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S009286742031254X">Integrated Morphoelectric and Transcriptomic Classification of Cortical GABAergic Cells</a>
Cell2020, <strong>all kinds of neurons!!!</strong></p>
<p><a href="https://elifesciences.org/articles/13222">Bicoid gradient formation and function in the Drosophila pre-syncytial blastoderm</a></p>
<p><a href="https://genesdev.cshlp.org/content/11/15/1963">An anteroposterior Dorsal gradient in theDrosophila embryo</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0960982213003333">Variation in the Dorsal Gradient Distribution Is a Source for Modified Scaling of Germ Layers in Drosophila</a></p>
<p><a href="https://www.nature.com/scitable/topicpage/gradient-based-dna-transcription-control-in-animals-1062/">Gradient-Based DNA Transcription Control in Animals</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2022.11.01.514680v1.abstract">Cell lineage predicts neural connectivity beyond cell type</a>
Konrad P. Kording</p>
<p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011574">Inferring causal connectivity from pairwise recordings and optogenetics</a>
Konrad P. Kording</p>
<p><a href="https://www.nature.com/articles/s41593-024-01806-0">Opportunities and challenges of single-cell and spatially resolved genomics methods for neuroscience discovery</a></p>
<p><a href="https://arxiv.org/pdf/2411.12091">Homogenized C. elegans Neural Activity and Connectivity Data</a></p>
<h3 id="gene-data">gene data</h3>
<p><a href="https://www.nature.com/articles/s41586-020-2879-3">Neuronal diversity and convergence in a visual system developmental atlas</a>
Claude Desplan NYU</p>
<p><a href="https://www.cell.com/cell/fulltext/S0092-8674(18)30720-7">A Single-Cell Transcriptome Atlas of the Aging Drosophila Brain</a></p>
<p><a href="https://www.nature.com/articles/s41586-021-04262-z">Decoding gene regulation in the fly brain</a></p>
<p><a href="https://www.nature.com/articles/s41586-021-04262-z">Decoding gene regulation in the fly brain</a>
<a href="https://www.nature.com/articles/s41586-022-04564-w">A complete temporal transcription factor series in the fly visual system</a>
Claude Desplan</p>
<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S001216062400160X">Isoforms of Terminal Selector Mamo Control Axon Guidance during Adult Drosophila Memory Center Construction via Semaphorin-1a</a></p>
<p><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03152-z">SURGE: uncovering context-specific genetic-regulation of gene expression from single-cell RNA sequencing using latent-factor models</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.05.07.593018v1"><strong>Benchmarking methods for mapping functional connectivity in the brain</strong></a></p>
<p><a href="https://www.nature.com/articles/s42003-023-04588-6">Cross-platform normalization enables machine learning model training on microarray and RNA-seq data simultaneously</a></p>
<p><a href="https://www.nature.com/articles/s41593-024-01806-0">Opportunities and challenges of single-cell and spatially resolved genomics methods for neuroscience discovery</a>
two new axes of biological variation: cell-intrinsic regulation of cell states and expression programs and interactions between cells.</p>
<p>nature neuroscience review 2024</p>
<p><a href="https://elifesciences.org/articles/27041">Science Forum: The Human Cell Atlas</a></p>
<h4 id="logical-gene-for-connectome">logical gene for connectome</h4>
<p><a href="https://arxiv.org/abs/2006.11287">Discovering Symbolic Models from Deep Learning with Inductive Biases</a></p>
<p><a href="https://www.nature.com/articles/s42254-022-00518-3">On scientific understanding with artificial intelligence</a></p>
<p><a href="https://arxiv.org/abs/2204.10532">End-to-end symbolic regression with transformers</a></p>
<p><a href="https://arxiv.org/abs/2305.01582">nterpretable Machine Learning for Science with PySR and SymbolicRegression.jl</a></p>
<p><a href="https://arxiv.org/abs/2404.19756">KAN: Kolmogorov-Arnold Networks</a></p>
<h4 id="gene-regulatory-network">gene regulatory network</h4>
<!-- [Inferring transcription factor regulatory networks from single-cell ATAC-seq data based on graph neural networks](https://www.nature.com/articles/s42256-022-00469-5) -->
<!-- [Gene regulatory network inference in the era of single-cell multi-omics](https://www.nature.com/articles/s41576-023-00618-5) -->
<p><a href="https://www.nature.com/articles/s41576-023-00618-5">Gene regulatory network inference in the era of single-cell multi-omics</a>Julio Saez-Rodriguez
Nature Reviews Genetics</p>
<p><a href="https://www.nature.com/articles/s41586-021-04262-z">Decoding gene regulation in the fly brain</a></p>
<p><a href="https://www.nature.com/articles/s42256-022-00469-5">Inferring transcription factor regulatory networks from single-cell ATAC-seq data based on graph neural networks</a></p>
<p><a href="https://www.nature.com/articles/s41587-024-02182-7">Inferring gene regulatory networks from single-cell multiome data using atlas-scale external data</a></p>
<p><a href="https://elifesciences.org/articles/19942">Genome-wide identification of neuronal activity-regulated genes in Drosophila</a></p>
<h4 id="application">application</h4>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.03.16.435261v2">Encoding innate ability through a genomic bottleneck</a></p>
<p><a href="https://arxiv.org/abs/2407.15908">The Genomic Code: The genome instantiates a generative model of the organism</a></p>
<h3 id="spiking-neural-network">Spiking neural network</h3>
<p><a href="https://openreview.net/pdf?id=8IvW2k5VeA">Exploring Loss Functions for Time-based Training Strategy in Spiking Neural Networks</a></p>
<h3 id="functional-and-structure-conenctome">functional and structure conenctome</h3>
<p><a href="https://www.nature.com/articles/nrn2575">Complex brain networks: graph theoretical analysis of structural and functional systems</a>
Ed Bullmore &amp; Olaf Sporns (2009).<br>
Nature reviews neuroscience</p>
<p>Engel, A. K., Gerloff, C., Hilgetag, C. C., &amp; Nolte, G. (2013). Intrinsic coupling modes: multiscale interactions in ongoing brain activity. Neuron, 80(4), 867-886.</p>
<p>Biswal, B., F. Zerrin Yetkin, Victor M. Haughton, James S. Hyde (1995). Functional connectivity in the motor cortex of resting human brain using echo-planar mri. Magnetic Resonance in Medicine 34(4): 537-541.</p>
<p>Markov NT, et al. (2014) A weighted and directed interareal connectivity matrix for macaque cerebral cortex. Cereb Cortex 24:17–36.</p>
<p>Donahue, C.J., Sotiropoulos, S.N., Jbabdi, S., Hernandez-Fernandez, M., Behrens, T.E., Dyrby, T.B., Coalson, T., Kennedy, H., Knoblauch, K., Van Essen, D.C., Glasser, M.F. (2016). Using diffusion tractography to predict cortical connection strength and distance: a quantitative comparison with tracers in the monkey. J. Neurosci. 36, 6758–6770. doi:10.1523/JNEUROSCI.0493-16.2016.</p>
<p>Honey, C., Thivierge JP, Sporns O (2010). Can structure predict function in the human brain? Neuroimage 52(3): 766-776.
Deco, G., McIntosh, A. R., Shen, K., Hutchison, R. M., Menon, R. S., Everling, S., Hagmann, P., &amp; Jirsa, V. K. (2014). Identification of optimal structural connectivity using functional connectivity and neural modeling. The Journal of neuroscience : the official journal of the Society for Neuroscience, 34(23), 7910–7916.</p>
<p>Dawson, DA, Cha K, Lewis LB, Mendola JD, Shmuel A (2013). &quot;Evaluation and calibration of functional network modeling methods based on known anatomical connections.&quot; Neuroimage 67: 331-343.</p>
<p>Predicting brain structural network using functional connectivity. Medical image analysis, 79, 102463. <a href="https://doi.org/10.1016/j.media.2022.102463">https://doi.org/10.1016/j.media.2022.102463</a>.</p>
<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S1361841522001104">Predicting brain structural network using functional connectivity</a>j.media<br>
multi-GCN based GAN, AD</p>
<h3 id="connectome-visualization">connectome visualization</h3>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.07.26.550598v1">CAVE: Connectome Annotation Versioning Engine</a>
<a href="https://vcg.seas.harvard.edu/publications/cave">https://vcg.seas.harvard.edu/publications/cave</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.10.05.561103v1">SmartEM: machine-learning guided electron microscopy</a></p>
<p>BOOK: <a href="https://link.springer.com/chapter/10.1007/978-1-4471-6497-5_21">Visualization in Connectomics</a></p>
<p><a href="https://www.pnas.org/doi/full/10.1073/pnas.1103583108">A Unified attentional bottleneck in the human brain</a>
connection bottleneck</p>
<p><a href="https://vcg.seas.harvard.edu/publications/cell2cell">Cell2Cell: Explorative Cell Interaction Analysis in Multi-Volumetric Tissue Data</a></p>
<p><a href="https://www.nature.com/articles/s41598-022-07959-x">Computation and visualization of cell–cell signaling topologies in single-cell systems data using Connectome</a></p>
<p><strong>human connectome</strong></p>
<ul>
<li><a href="https://doi.org/10.1162/netn_a_00044">NeuroCave: A web-based immersive visualization platform for exploring connectome datasets</a></li>
</ul>
<h2 id="graph">graph</h2>
<p>JC: <a href="https://docs.google.com/spreadsheets/d/10R6poOIIqA4OCsiSGs5mK4Zkq9pm8aPxk1IQ3Zl0NPU">https://docs.google.com/spreadsheets/d/10R6poOIIqA4OCsiSGs5mK4Zkq9pm8aPxk1IQ3Zl0NPU</a></p>
<p>DGL 2.0: Streamlining Your GNN Data Pipeline from Bottleneck to Boost
<a href="https://www.dgl.ai/">https://www.dgl.ai/</a>
<a href="https://github.com/dmlc/dgl/blob/master/examples/README.md#agnn">https://github.com/dmlc/dgl/blob/master/examples/README.md#agnn</a></p>
<p><a href="https://ieeexplore.ieee.org/document/4700287">The Graph Neural Network Model</a></p>
<p><a href="https://arxiv.org/abs/1611.07308">Variational Graph Auto-Encoders</a>
Bayesian Deep Learning Workshop (NIPS 2016)</p>
<p>code: <a href="https://github.com/DaehanKim/vgae_pytorch">https://github.com/DaehanKim/vgae_pytorch</a></p>
<p><a href="https://arxiv.org/abs/1710.10903">Graph Attention Networks</a>
Yoshua Bengio</p>
<p><a href="https://petar-v.com/GAT/">Graph Attention Networks</a></p>
<p><a href="https://arxiv.org/abs/1803.03735">Attention-based Graph Neural Network for Semi-supervised Learning</a></p>
<h3 id="link-prediction">Link prediction</h3>
<p><a href="https://paperswithcode.com/task/link-prediction">https://paperswithcode.com/task/link-prediction</a></p>
<p>GraphSAGE: <a href="https://arxiv.org/abs/1706.02216v4">Inductive Representation Learning on Large Graphs</a>
GraphSAGE, a general, inductive framework that leverages node feature information (e.g., text attributes) to efficiently generate node embeddings for previously unseen data.</p>
<h3 id="graph-embedding">Graph Embedding</h3>
<p><a href="https://arxiv.org/abs/1503.03578">LINE: Large-scale Information Network Embedding</a></p>
<p><a href="">GraphVite - graph embedding at high speed and large scale</a></p>
<h3 id="review">review</h3>
<p><a href="https://arxiv.org/abs/1901.00596">A Comprehensive Survey on Graph Neural Networks</a>
Cited by 11155</p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S2666651021000012">Graph neural networks: A review of methods and applications</a>
Cited by 6966</p>
<p><a href="https://ieeexplore.ieee.org/abstract/document/9416834">Graph Learning: A Survey</a></p>
<p><a href="https://paperswithcode.com/methods/category/graph-models">papers with code: Graph Models</a></p>
<h3 id="disease-gene">disease gene</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0925231221011012">GANLDA: Graph attention network for lncRNA-disease associations prediction</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2022.11.21.517436v1">Genome-wide prediction of dominant and recessive neurodevelopmental disorder risk genes</a></p>
<p><a href="https://www.cell.com/ajhg/fulltext/S0002-9297(20)30108-7">Mantis-ml: Disease-Agnostic Gene Prioritization from High-Throughput Genomic Screens by Stochastic Semi-supervised Learning</a></p>
<h2 id="neuroai">NeuroAI</h2>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(22)00806-6#%20">In vitro neurons learn and exhibit sentience when embodied in a simulated game-world</a>
<a href="https://www.nature.com/articles/s41928-023-01069-w">Brain organoid reservoir computing for artificial intelligence</a></p>
<p><a href="https://arxiv.org/abs/2212.13345">The Forward-Forward Algorithm: Some Preliminary Investigations</a>
Geoffrey Hinton
<a href="https://www.sciencedirect.com/science/article/pii/S0959438818302174">Constraining computational models using electron microscopy wiring diagrams</a>
Ashok Litwin-Kumar , Srinivas C Turaga
Current Opinion in Neurobiology, 2019</p>
<h3 id="connectome--ann">Connectome &amp; ANN</h3>
<p><a href="connectome%20inspired%20neural%20network.html">connectome inspired neural network</a><br>
html <a href="https://lengyuner.github.io/blog/connectome%20inspired%20neural%20network.html">connectome inspired neural network</a></p>
<h3 id="geometry">geometry</h3>
<p><a href="https://openreview.net/forum?id=dqWobzlAGb">Modelling brain connectomes networks: Solv is a worthy competitor to hyperbolic geometry!</a>
ICLR 2024 Conference Submission</p>
<h3 id="brain-simulation--activity-prediction">Brain simulation / activity prediction</h3>
<p><a href="https://openreview.net/forum?id=7ntI4kcoqG">AMAG: Additive, Multiplicative and Adaptive Graph Neural Network For Forecasting Neuron Activity</a>
NeurIPS 2023 poster</p>
<h4 id="model">model</h4>
<p><a href="https://arxiv.org/abs/2002.09405">Learning to Simulate Complex Physics with Graph Networks</a>
为了做这个，我们需要什么样的data。有wide field and the 2p microscope，是不是就可以模拟一些基础规律了</p>
<h4 id="whole-brain-imaging-data">whole brain imaging data</h4>
<p><a href="https://www.nature.com/articles/s41467-023-41261-2">The spatial and temporal structure of neural activity across the fly brain</a> swept confocally-aligned planar excitation (SCAPE) microscopy,
1419 ± 78 stable, single-cell ROIs per animal, contained nearly all neurons within 70 μm of the dorsal surface</p>
<h3 id="network-analysis">Network analysis</h3>
<p>BOOK <a href="https://www.sciencedirect.com/book/9780124079083/fundamentals-of-brain-network-analysis">Fundamentals of Brain Network Analysis</a></p>
<p>BOOK <a href="https://www.sciencedirect.com/science/article/pii/B9780124079083000030">Chapter 3 - Connectivity Matrices and Brain Graphs</a></p>
<p>Review <a href="https://www.frontiersin.org/articles/10.3389/fnsys.2020.609316/full">Neural Networks With Motivation</a></p>
<p><a href="https://elifesciences.org/articles/04250">Automatic discovery of cell types and microcircuitry from neural connectomics</a>
Eric Jonas, Konrad Kording</p>
<h3 id="synapse-plasticity">synapse plasticity</h3>
<h4 id="stdp">STDP</h4>
<p>nearest neighbor STDP (Izhikevich, 2003)
<a href="https://doi.org/10.1162/089976603321891783">Relating STDP to BCM</a></p>
<p><a href="https://arxiv.org/abs/2310.20309">Tensor formalism for predicting synaptic connections with ensemble modeling or optimization</a>
Tirthabir Biswas, Tianzhi Lambus Li, James E. Fitzgerald</p>
<h2 id="neural-data-recording">neural data recording</h2>
<h3 id="whole-brain-imaging">whole brain imaging</h3>
<p><a href="https://www.nature.com/articles/s41467-023-41261-2">The spatial and temporal structure of neural activity across the fly brain</a>
Evan S. Schaffer ... L. F. Abbott &amp; Richard Axel</p>
<p><a href="https://www.nature.com/articles/s42254-022-00430-w?fromPaywallRec=false">Imaging whole-brain activity to understand behaviour</a>
Review Article</p>
<p><a href="https://www.science.org/doi/full/10.1126/science.adf9725">Wide-field fluorescence lifetime imaging of neuron spiking and subthreshold activity in vivo</a></p>
<p><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(23)01763-3">Mapping the neural dynamics of locomotion across the Drosophila brain</a></p>
<p><a href="https://www.pnas.org/doi/full/10.1073/pnas.0905267106">Correspondence of the brain's functional architecture during activation and rest</a></p>
<p><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(17)30813-8">Whole-Brain Calcium Imaging Reveals an Intrinsic Functional Network in Drosophila</a>
Current Bilogy</p>
<p><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2006732">Fast near-whole–brain imaging in adult Drosophila during responses to stimuli and behavior</a>
Plos Biology</p>
<p><a href="https://www.cell.com/iscience/fulltext/S2589-0042(19)30091-4">Imaging through the Whole Brain of Drosophila at λ/20 Super-resolution</a></p>
<p><a href="https://www.nature.com/articles/s41586-021-03497-0">Coupling of activity, metabolism and behaviour across the Drosophila brain</a> Surya Ganguli &amp; Thomas R. Clandinin</p>
<h3 id="cell-tracking">cell tracking</h3>
<p><a href="https://www.biorxiv.org/content/10.1101/2020.06.22.165696v3.full">EMC2: A versatile algorithm for robust tracking of calcium dynamics from individual neurons in behaving animals</a></p>
<p><a href="https://elifesciences.org/articles/59187">3DeeCellTracker, a deep learning-based pipeline for segmenting and tracking cells in 3D time lapse images</a></p>
<h3 id="wormid">wormid</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S2667237524003540">Unifying community whole-brain imaging datasets enables robust neuron identification and reveals determinants of neuron position in C. elegans</a>
Daniel Y. Sprague,..., Saul Kato</p>
<h2 id="neural-data-analysis">neural data analysis</h2>
<p><a href="https://direct.mit.edu/neco/article/27/1/104/8035/Spatiotemporal-Conditional-Inference-and">Spatiotemporal conditional inference and hypothesis tests for neural ensemble spiking precision</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0959438816302689">Maximum entropy models as a tool for building precise neural controls</a>
Cristina Savin, Gašper Tkačik</p>
<h3 id="reivew">reivew</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0959438817302428">Neural data science: accelerating the experiment-analysis-theory cycle in large-scale neuroscience </a>
ICCV, 2019</p>
<p><a href="https://www.science.org/doi/full/10.1126/science.adp7429">Analysis methods for large-scale neuronal recordings</a>
Carsen Stringer, Marius Pachitariu</p>
<h3 id="high-dimensional-data">high dimensional data</h3>
<p><a href="https://www.science.org/doi/10.1126/science.aav7893">Spontaneous behaviors drive multidimensional, brainwide activity</a>
Carsen Stringer</p>
<p><a href="https://www.nature.com/articles/s41586-019-1346-5">High-dimensional geometry of population responses in visual cortex</a>
Carsen Stringer</p>
<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0896627324001211">Simultaneous, cortex-wide dynamics of up to <strong>1 million neurons</strong> reveal unbounded scaling of dimensionality with neuron number</a></p>
<h3 id="dimensional-reduction">dimensional reduction</h3>
<p>Phate: <a href="https://www.nature.com/articles/s41587-019-0336-3">Visualizing structure and transitions in high-dimensional biological data</a>
Phate</p>
<h3 id="manifold-reconstriction">manifold reconstriction</h3>
<p><a href="https://arxiv.org/abs/2012.13804">Approximation of Functions on Manifolds in High Dimension from Noisy Scattered Data</a>
<strong>Manifold Locally Optimal Projection (MLOP)</strong></p>
<p><a href="https://arxiv.org/abs/2102.01750">Manifold Repairing, Reconstruction and Denoising from Scattered Data in High-Dimension</a>
<strong>Repairing Manifold Locally Optimal Projection (R-MLOP)</strong></p>
<p><a href="https://www.cs.tau.ac.il/~dcor/articles/2007/Parameterization-free-Projection.pdf">Parameterization-free Projection for Geometry Reconstruction</a></p>
<h3 id="activity-analysis">activity analysis</h3>
<p><a href="https://www.nature.com/articles/s41586-023-06683-4">Neural signal propagation atlas of Caenorhabditis elegans</a> Andrew M. Leifer</p>
<p><a href="https://www.nature.com/articles/s41593-024-01845-7">Dynamical constraints on neural population activity</a>
Byron M. Yu &amp; Aaron P. Batista</p>
<p><a href="https://www.nature.com/articles/s41586-023-06714-0">Preserved neural dynamics across animals performing similar behaviour</a></p>
<p><a href="https://www.nature.com/articles/s41593-024-01845-7">Dynamical constraints on neural population activity</a>
Byron M. Yu &amp; Aaron P. Batista</p>
<p>[]</p>
<h2 id="encoding-and-decoding">encoding and decoding</h2>
<p><a href="https://www.science.org/doi/10.1126/science.1234330">Neural Decoding of Visual Imagery During Sleep</a></p>
<p><a href="https://www.jneurosci.org/content/37/14/3764.abstract">Locomotion Enhances Neural Encoding of Visual Stimuli in Mouse V1</a></p>
<p><a href="https://journals.physiology.org/doi/full/10.1152/jn.01046.2012">Bayesian decoding using unsorted spikes in the rat hippocampus</a>
Fabian Kloosterman, Stuart P. Layton, Zhe Chen, and Matthew A. Wilson</p>
<p><a href="https://www.cell.com/cell/fulltext/S0092-8674(24)00980-2">Decoding the brain: From neural representations to mechanistic models</a></p>
<h3 id="deep-learning-method">deep learning method</h3>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.09.18.558113v1">Neural Data Transformer 2: Multi-context Pretraining for Neural Spiking Activity</a></p>
<p>LFADS: <a href="https://www.nature.com/articles/s41592-018-0109-9">Inferring single-trial neural population dynamics using sequential auto-encoders</a><br>
<a href="https://arxiv.org/abs/1608.06315">LFADS (method part)</a></p>
<p>MtM: <a href="https://openreview.net/forum?id=nRRJsDahEg">Towards a &quot;Universal Translator&quot; for Neural Dynamics at Single-Cell, Single-Spike Resolution</a>
Yizi, Liam, Cole</p>
<p>NEMO: <a href="https://www.biorxiv.org/content/10.1101/2024.11.05.622159v1">In vivo cell-type and brain region classification via multimodal contrastive learning</a></p>
<p><a href="https://poyo-brain.github.io/">POYO-1: A Unified, Scalable Framework for Neural Population Decoding</a></p>
<h4 id="">?</h4>
<p><a href="https://openreview.net/forum?id=90O5cvFZkZ">GUST: Combinatorial Generalization by Unsupervised Grouping with Neuronal Coherence</a>
NeurIPS 2023 poster</p>
<h2 id="behavior">behavior</h2>
<p>GLM-HMM: <a href="https://www.nature.com/articles/s41593-019-0533-x">Unsupervised identification of the internal states that shape natural behavior</a><br>
<a href="https://www.nature.com/articles/s41593-021-01007-z">Mice alternate between discrete strategies during perceptual decision-making</a>
Jonathan W. Pillow</p>
<p><a href="https://arxiv.org/abs/2007.08723">End-to-end Deep Prototype and Exemplar Models for Predicting Human Behavior</a>
Thomas L. Griffiths</p>
<p><a href="https://www.frontiersin.org/articles/10.3389/fncir.2012.00108/full">Spatial vision in insects is facilitated by shaping the dynamics of visual input through behavioral action</a></p>
<p><a href="https://www.nature.com/articles/nature14855">Arithmetic and local circuitry underlying dopamine prediction errors</a></p>
<p><a href="https://www.science.org/doi/10.1126/science.275.5306.1593">A Neural Substrate of Prediction and Reward</a></p>
<p><a href="https://openreview.net/forum?id=nosngu5XwY9">Dynamic Inverse Reinforcement Learning for Characterizing Animal Behavior</a>  Zoe Ashwood, Aditi Jha, Jonathan W. Pillow<br>
NeurIPS 2022</p>
<p><a href="https://www.nature.com/articles/s41586-022-05611-2">Spontaneous behaviour is structured by reinforcement without explicit reward</a>
Sandeep Robert Datta</p>
<h3 id="oscilation">oscilation</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627323002143">Brain rhythms have come of age</a></p>
<h2 id="vision">vision</h2>
<p><a href="https://www.pnas.org/doi/full/10.1073/pnas.1922294117">The limits of color awareness during active, real-world vision</a>
using VR to test the color fade perception</p>
<p><a href="https://www.frontiersin.org/articles/10.3389/fncir.2015.00033/full">A common evolutionary origin for the ON- and OFF-edge motion detection pathways of the Drosophila visual system</a>
Ian A. Meinertzhagen</p>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.12.20.473513v1">Neuronal circuits integrating visual motion information in Drosophila</a>
T4 and T5 neuron</p>
<p><a href="https://www.nature.com/articles/s41586-024-08255-6">Social state alters vision using three circuit mechanisms in Drosophila</a>
Vanessa Ruta &amp; Gerald M. Rubin</p>
<h3 id="drosophila">drosophila</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627308007058">The Neural Substrate of Spectral Preference in Drosophila</a></p>
<h2 id="others">others</h2>
<h3 id="hippocamus">hippocamus</h3>
<p><a href="https://www.jneurosci.org/content/28/52/14271.short">Network Dynamics Underlying the Formation of Sparse, Informative Representations in the Hippocampus</a></p>
<p><a href="https://www.pnas.org/doi/abs/10.1073/pnas.1408365111">Sparse and distributed coding of episodic memory in neurons of the human hippocampus</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0166432819303936">Synaptic properties of newly generated granule cells support sparse coding in the adult hippocampus</a></p>
<h3 id="neural-type">neural type</h3>
<p><a href="https://www.nature.com/articles/s41586-023-06808-9">Molecularly defined and spatially resolved cell atlas of the whole mouse brain</a>
Xiaowei Zhuang</p>
<p><a href="https://www.nature.com/articles/s41592-019-0631-4">Probabilistic cell typing enables fine mapping of closely related cell types in situ</a></p>
<h3 id="gene">gene</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867416300034">Bayesian Sparse Regression Analysis Documents the Diversity of Spinal Inhibitory Interneurons</a></p>
<p><a href="https://arxiv.org/abs/2306.15794">HyenaDNA: Long-Range Genomic Sequence Modeling at Single Nucleotide Resolution</a>   Transformer + gene</p>
<p><a href="https://www.science.org/doi/10.1126/science.1202529">Synaptic Pruning by Microglia Is Necessary for Normal Brain Development</a></p>
<h3 id="gene-normalization">gene normalization</h3>
<p><a href="https://scanpy.readthedocs.io/en/stable/">Scanpy – Single-Cell Analysis in Python</a>
<a href="https://scanpy.readthedocs.io/en/stable/tutorials/basics/clustering.html">scanpy Preprocessing and clustering</a>
<a href="https://scanpy.readthedocs.io/en/stable/generated/scanpy.pp.highly_variable_genes.html">scanpy highly_variable_genes</a></p>
<p><a href="https://satijalab.org/seurat/reference/normalizedata">seurat Normalize Data</a></p>
<p><a href="https://bioconductor.org/packages/release/workflows/vignettes/RNAseq123/inst/doc/limmaWorkflow.html">RNA-seq analysis is easy as 1-2-3 with limma, Glimma and edgeR</a></p>
<p><a href="https://www.nature.com/articles/nbt.4096">Integrating single-cell transcriptomic data across different conditions, technologies, and species</a></p>
<h3 id="morphology">morphology</h3>
<p><a href="https://journals.physiology.org/doi/full/10.1152/jn.00865.2006">Mapping Function Onto Neuronal Morphology</a>
Klaus M. Stiefel1, andTerrence J. Sejnowski</p>
<p><a href="https://www.cell.com/fulltext/S0896-6273(04)00498-2">Synaptic Connectivity and Neuronal Morphology: Two Sides of the Same Coin</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0165027010006758">Automated analysis of neuronal morphology, synapse number and synaptic recruitment</a></p>
<p><a href="https://www.nature.com/articles/nprot.2008.51">L-Measure: a web-accessible tool for the analysis, comparison and search of digital reconstructions of neuronal morphologies</a></p>
<p><a href="https://openreview.net/forum?id=p6hArCtwLAU">TreeMoCo: Contrastive Neuron Morphology Representation Learning</a></p>
<h4 id="maximum-weighted-independent-set-mwis">maximum weighted independent set (MWIS)</h4>
<p><a href="https://arxiv.org/abs/1810.10834">Exactly Solving the Maximum Weight Independent Set Problem on Large Real-World Graphs</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0166218X02002056">A note on greedy algorithms for the maximum weighted independent set problem</a></p>
<p>Graph compression or sparsification
<a href="https://arxiv.org/pdf/2007.07862.pdf">Vertex Sparsification for Edge Connectivity</a></p>
<h4 id="maximum-dispersion">Maximum Dispersion</h4>
<p>Maximum Diversity Problem (MDP)
<a href="https://arxiv.org/abs/1706.09185">Dispersion on Trees</a>
<a href="https://ieeexplore.ieee.org/document/7004250">Estimating Pairwise Distances in Large Graphs</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0196677400911453">Approximation Algorithms for Dispersion Problems</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0167637797000345">Approximation Algorithms for Maximum Dispersion</a>
<a href="https://epubs.siam.org/doi/10.1137/0404048">Obnoxious Facility Location on Graphs</a></p>
<p><a href="https://zhuanlan.zhihu.com/p/101724381">变邻域搜索(VNS)算法求解Max-Mean Dispersion Problem</a></p>
<p><a href="https://arxiv.org/abs/1503.00980">On memetic search for the max-mean dispersion problem</a>
<a href="https://www.sciencedirect.com/science/article/pii/S0950705115001860">Tabu search for the Max–Mean Dispersion Problem</a></p>
<!-- [Exact Solution of P-dispersion Problems](https://www.semanticscholar.org/paper/Exact-Solution-of-P-dispersion-Problems-Pisinger/1eb3810077c0af9d46ed5ff2b0819d954c97dcae) -->
<p><a href="https://www.sciencedirect.com/science/article/pii/S0305054804002552">Upper bounds and exact algorithms for p-dispersion problems
</a>
<a href="http://yalma.fime.uanl.mx/~roger/work/teaching/class_tso/docs_project/problems/PDP/cor-2006-Pisinger.pdf">http://yalma.fime.uanl.mx/~roger/work/teaching/class_tso/docs_project/problems/PDP/cor-2006-Pisinger.pdf</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0377221721006548">A review on discrete diversity and dispersion maximization from an OR perspective</a></p>
<p><a href="https://pubsonline.informs.org/doi/abs/10.1287/opre.42.2.299">Heuristic and special case algorithms for dispersion problems</a></p>
<p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/net.3230120206">A network flow solution to some nonlinear 0-1 programming problems, with applications to graph theory</a></p>
<h3 id="place-cell-and-grid-cell">place cell and grid cell</h3>
<p><a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/medial-entorhinal-cortex">medial entorhinal cortex</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S009286742031388X">The Tolman-Eichenbaum Machine: Unifying Space and Relational Memory through Generalization in the Hippocampal Formation</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.10.28.466284v5">Emergence of robust global modules from local interactions and smooth gradients</a></p>
<h3 id="cognition">cognition</h3>
<h4 id="attention">attention</h4>
<h4 id="learning-and-memory">learning and memory</h4>
<p><a href="https://journals.physiology.org/doi/full/10.1152/jn.00454.2022">How our understanding of memory replay evolves</a></p>
<h4 id="decision-making">decision making</h4>
<p><a href="https://www.jneurosci.org/content/28/12/3017">Bounded Integration in Parietal Cortex Underlies Decisions Even When Viewing Duration Is Dictated by the Environment</a></p>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(23)00758-4">Dopamine pathways mediating affective state transitions after sleep loss</a></p>
<h3 id="ds">DS</h3>
<h4 id="quadratic-assignment-problem">Quadratic assignment problem</h4>
<p><a href="https://en.wikipedia.org/wiki/Quadratic_assignment_problem">quadratic assignment problem</a></p>
<p><a href="https://arxiv.org/abs/1905.10124">Sliced Gromov-Wasserstein</a></p>
<p><a href="https://pythonot.github.io/gen_modules/ot.gromov.html#module-ot.gromov">ot.gromov</a>
Fused Gromov-Wasserstein transport</p>
<h4 id="similarity-matrix">similarity matrix</h4>
<p><a href="https://arxiv.org/abs/1502.07541">Euclidean Distance Matrices: Essential Theory, Algorithms and Applications</a></p>
<p><a href="https://www.frontiersin.org/articles/10.3389/neuro.06.004.2008/full">Representational similarity analysis – connecting the branches of systems neuroscience</a></p>
<p><a href="https://arxiv.org/abs/1502.07541">Euclidean Distance Matrices: Essential Theory, Algorithms and Applications</a></p>
<p><strong>CCA</strong> <a href="https://www.worldscientific.com/doi/abs/10.1142/S012906570000034X">Kernel and nonlinear canonical correlation analysis</a></p>
<p><a href="https://arxiv.org/abs/1905.00414">Similarity of Neural Network Representations Revisited</a>
Simon Kornblith, Mohammad Norouzi, Honglak Lee, Geoffrey Hinton</p>
<p><a href="https://arxiv.org/abs/2211.11665">Representational dissimilarity metric spaces for stochastic neural networks</a>
Lyndon R. Duong, Jingyang Zhou, Josue Nassar, Jules Berman, Jeroen Olieslagers, Alex H. Williams</p>
<h3 id="ml">ML</h3>
<p><a href="https://arxiv.org/abs/2305.14608">Inverse Reinforcement Learning with the Average Reward Criterion</a></p>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(20)30705-4">Artificial Neural Networks for Neuroscientists: A Primer</a></p>
<p><a href="https://arxiv.org/abs/1706.03762">Attention Is All You Need</a>
<strong>Transformer</strong></p>
<p><a href="https://arxiv.org/abs/1506.04214">Convolutional LSTM Network: A Machine Learning Approach for Precipitation Nowcasting</a>
recurrent CNN</p>
<h4 id="nas">NAS</h4>
<p><a href="https://arxiv.org/abs/1902.09635">NAS-Bench-101: Towards Reproducible Neural Architecture Search</a>
Chris Ying, Aaron Klein, Esteban Real, Eric Christiansen, Kevin Murphy, Frank Hutter</p>
<h4 id="cnn-kernel">CNN kernel</h4>
<p>Clustering convolutional kernels</p>
<p><a href="https://www.worldscientific.com/doi/epdf/10.1142/S0219467819500190">Framework for Automatic Selection of Kernels based on Convolutional Neural Networks and CkMeans Clustering Algorithm</a></p>
<p><a href="https://arxiv.org/abs/1804.11191">How convolutional neural network see the world - A survey of convolutional neural network visualization methods</a></p>
<p><a href="https://arxiv.org/abs/1311.2901">Visualizing and Understanding Convolutional Networks</a></p>
<h3 id="sparse-decoding">sparse decoding</h3>
<p>Identifiable Deep Generative Models via Sparse Decoding
<a href="https://arxiv.org/abs/2110.10804">https://arxiv.org/abs/2110.10804</a></p>
<p>UFLDL Tutorial
Sparse Coding
<a href="http://ufldl.stanford.edu/tutorial/unsupervised/SparseCoding/">http://ufldl.stanford.edu/tutorial/unsupervised/SparseCoding/</a></p>
<p>Sensing Matrix Optimization for Block-Sparse Decoding
<a href="https://ieeexplore.ieee.org/abstract/document/5872076">https://ieeexplore.ieee.org/abstract/document/5872076</a></p>
<p>A Tutorial on Decoding Techniques of Sparse Code Multiple Access
<a href="https://ieeexplore.ieee.org/abstract/document/9782313">https://ieeexplore.ieee.org/abstract/document/9782313</a></p>
<p>On the iterative decoding of sparse quantum codes
<a href="https://arxiv.org/abs/0801.1241">https://arxiv.org/abs/0801.1241</a></p>
<h3 id="others-1">others</h3>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.01.27.577215v1">Mapping the developmental structure of stereotyped and individual-unique behavioral spaces in C. elegans</a></p>
<p><a href="https://arxiv.org/abs/1910.01312">A sparse semismooth Newton based augmented Lagrangian method for large-scale support vector machines</a> SVM</p>
<p><a href="https://www.annualreviews.org/doi/abs/10.1146/annurev.neuro.31.060407.125555">Vestibular System: The Many Facets of a Multimodal Sense</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627321007054">Spatial modulation of hippocampal activity in freely moving macaques</a>
Dora E. Angelaki</p>
<p><a href="https://www.brains-explained.com/how-hubel-and-wiesel-revolutionized-neuroscience/">How Hubel and Wiesel Revolutionized Neuroscience and Made Me a Neuroscientist</a>
a story</p>
<p><a href="https://www.nature.com/articles/s41583-023-00764-z">Sleep and circadian rhythmicity as entangled processes serving homeostasis</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627324000473">Tuned geometries of hippocampal representations meet the computational demands of social memory</a></p>
<p><a href="https://www.nature.com/articles/s41467-023-37980-1">Complex computation from developmental priors</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1053811919300114">A practical guide to linking brain-wide gene expression and neuroimaging data</a>
gene</p>
<h3 id="wiki">wiki</h3>
<p>wiki <a href="https://en.wikipedia.org/wiki/Non-negative_least_squares">Non-negative least squares</a></p>
<p>wiki <a href="https://en.wikipedia.org/wiki/Kronecker_product">Kronecker product</a></p>
<p>wiki <a href="https://en.wikipedia.org/wiki/F-score">F-score</a></p>
<p>wiki <a href="https://en.wikipedia.org/wiki/Precision_and_recall">Precision_and_recall</a></p>
<h1 id="todo">TODO</h1>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s41586-022-05562-8">Synaptic gradients transform object location to action</a>
S. Lawrence Zipursky &amp; Gwyneth M. Card</p>
</li>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.adh7814">Hebbian instruction of axonal connectivity by endogenous correlated spontaneous activity</a></p>
</li>
<li></li>
<li>
<p><a href="https://www.nature.com/articles/s41467-024-50608-2">Whole brain alignment of spatial transcriptomics between humans and mice with BrainAlign</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.09.11.557287v3">Cell2Sentence: Teaching Large Language Models the Language of Biology</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/nmeth.4402">Reversed graph embedding resolves complex single-cell trajectories</a></p>
</li>
<li>
<p><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4772-0">Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2002.04461">TrajectoryNet: A Dynamic Optimal Transport Network for Modeling Cellular Dynamics</a></p>
</li>
<li>
<p><a href="https://academic.oup.com/nar/article/50/15/e86/6593121?login=false">Inference of cell state transitions and cell fate plasticity from single-cell with MARGARET</a></p>
</li>
</ul>

            
            
        </body>
        </html>