<!DOCTYPE html>
        <html>
        <head>
            <meta charset="UTF-8">
            <title>paper reading list</title>
            <style>
/* From extension vscode.github */
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

.vscode-dark img[src$=\#gh-light-mode-only],
.vscode-light img[src$=\#gh-dark-mode-only],
.vscode-high-contrast:not(.vscode-high-contrast-light) img[src$=\#gh-light-mode-only],
.vscode-high-contrast-light img[src$=\#gh-dark-mode-only] {
	display: none;
}

</style>
            <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex/dist/katex.min.css">
<link href="https://cdn.jsdelivr.net/npm/katex-copytex@latest/dist/katex-copytex.min.css" rel="stylesheet" type="text/css">
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">
<style>
            body {
                font-family: -apple-system, BlinkMacSystemFont, 'Segoe WPC', 'Segoe UI', system-ui, 'Ubuntu', 'Droid Sans', sans-serif;
                font-size: 14px;
                line-height: 1.6;
            }
        </style>
        <style>
.task-list-item {
    list-style-type: none;
}

.task-list-item-checkbox {
    margin-left: -20px;
    vertical-align: middle;
    pointer-events: none;
}
</style>
<style>
:root {
  --color-note: #0969da;
  --color-tip: #1a7f37;
  --color-warning: #9a6700;
  --color-severe: #bc4c00;
  --color-caution: #d1242f;
  --color-important: #8250df;
}

</style>
<style>
@media (prefers-color-scheme: dark) {
  :root {
    --color-note: #2f81f7;
    --color-tip: #3fb950;
    --color-warning: #d29922;
    --color-severe: #db6d28;
    --color-caution: #f85149;
    --color-important: #a371f7;
  }
}

</style>
<style>
.markdown-alert {
  padding: 0.5rem 1rem;
  margin-bottom: 16px;
  color: inherit;
  border-left: .25em solid #888;
}

.markdown-alert>:first-child {
  margin-top: 0
}

.markdown-alert>:last-child {
  margin-bottom: 0
}

.markdown-alert .markdown-alert-title {
  display: flex;
  font-weight: 500;
  align-items: center;
  line-height: 1
}

.markdown-alert .markdown-alert-title .octicon {
  margin-right: 0.5rem;
  display: inline-block;
  overflow: visible !important;
  vertical-align: text-bottom;
  fill: currentColor;
}

.markdown-alert.markdown-alert-note {
  border-left-color: var(--color-note);
}

.markdown-alert.markdown-alert-note .markdown-alert-title {
  color: var(--color-note);
}

.markdown-alert.markdown-alert-important {
  border-left-color: var(--color-important);
}

.markdown-alert.markdown-alert-important .markdown-alert-title {
  color: var(--color-important);
}

.markdown-alert.markdown-alert-warning {
  border-left-color: var(--color-warning);
}

.markdown-alert.markdown-alert-warning .markdown-alert-title {
  color: var(--color-warning);
}

.markdown-alert.markdown-alert-tip {
  border-left-color: var(--color-tip);
}

.markdown-alert.markdown-alert-tip .markdown-alert-title {
  color: var(--color-tip);
}

.markdown-alert.markdown-alert-caution {
  border-left-color: var(--color-caution);
}

.markdown-alert.markdown-alert-caution .markdown-alert-title {
  color: var(--color-caution);
}

</style>
        
        </head>
        <body class="vscode-body vscode-light">
            <h1 id="paper-reading-list">paper reading list</h1>
<h2 id="table-of-contents">Table of Contents</h2>
<ul>
<li><a href="#paper-reading-list">paper reading list</a>
<ul>
<li><a href="#table-of-contents">Table of Contents</a></li>
<li><a href="#connectome">Connectome</a>
<ul>
<li><a href="#connectome-dataset">connectome dataset</a></li>
<li><a href="#technology-of-connectome">technology of connectome</a>
<ul>
<li><a href="#connection-reconsturction">connection reconsturction</a></li>
</ul>
</li>
<li><a href="#connectome-analysis">connectome analysis</a></li>
<li><a href="#gap-junction">gap junction</a></li>
<li><a href="#connectome-and-gene">connectome and gene</a>
<ul>
<li><a href="#rl-and-logical-rule">RL and logical rule</a></li>
<li><a href="#gene-data-of-drosophila">gene data of Drosophila</a></li>
<li><a href="#other-methods-to-reconstruct-connectome">other methods to reconstruct connectome</a></li>
<li><a href="#human-connectome">human connectome</a></li>
<li><a href="#experiment-support">experiment support</a></li>
<li><a href="#geneweave-application">GeneWeave application</a></li>
</ul>
</li>
<li><a href="#cell-cell-communication-and-interaction">cell-cell communication and interaction</a></li>
<li><a href="#protein-interaction">protein interaction</a>
<ul>
<li><a href="#experiment-support-1">experiment support</a></li>
<li><a href="#deep-learning-methods">deep learning methods</a></li>
<li><a href="#result-verification">result verification</a></li>
</ul>
</li>
<li><a href="#data-mapping-between-transcriptome-and-connectome">data mapping between transcriptome and connectome</a>
<ul>
<li><a href="#neural-type">neural type</a></li>
<li><a href="#neural-activity-and-connectivity">neural activity and connectivity</a></li>
</ul>
</li>
<li><a href="#morphology">Morphology</a>
<ul>
<li><a href="#morphology-data">morphology data</a></li>
<li><a href="#maximum-weighted-independent-set-mwis">maximum weighted independent set (MWIS)</a></li>
<li><a href="#maximum-dispersion">Maximum Dispersion</a></li>
</ul>
</li>
<li><a href="#connectome-visualization">connectome visualization</a></li>
<li><a href="#gene-data--transcriptome">gene data / transcriptome</a>
<ul>
<li><a href="#gene-data-for-drosophila">gene data for Drosophila</a></li>
<li><a href="#other-gene-dataset">other gene dataset</a></li>
<li><a href="#review">review</a></li>
<li><a href="#analysis">analysis</a></li>
<li><a href="#logical-gene-for-connectome">logical gene for connectome</a></li>
<li><a href="#gene-regulatory-network">gene regulatory network</a></li>
<li><a href="#review-paper">review paper</a></li>
<li><a href="#sota-models">sota models</a></li>
<li><a href="#application">application</a></li>
</ul>
</li>
<li><a href="#gene-normalization">gene normalization</a></li>
<li><a href="#functional-and-structure-conenctome">functional and structure conenctome</a></li>
<li><a href="#other-connectome-model">other connectome model:</a></li>
</ul>
</li>
<li><a href="#reinforcement-learning-rl">Reinforcement learning (RL)</a></li>
<li><a href="#graph">graph</a>
<ul>
<li><a href="#link-prediction">Link prediction</a>
<ul>
<li><a href="#review-1">review</a></li>
</ul>
</li>
<li><a href="#graph-embedding">Graph Embedding</a></li>
<li><a href="#review-2">review</a></li>
<li><a href="#"></a></li>
<li><a href="#disease-gene">disease gene</a></li>
<li><a href="#graph-and-connectome">graph and connectome</a></li>
<li><a href="#other">other</a></li>
<li><a href="#olfactory-system">olfactory system</a></li>
<li><a href="#mean-field">mean field</a></li>
</ul>
</li>
<li><a href="#neuroai">NeuroAI</a>
<ul>
<li><a href="#connectome--ann">Connectome &amp; ANN</a></li>
<li><a href="#geometry">geometry</a></li>
<li><a href="#brain-simulation--activity-prediction">Brain simulation / activity prediction</a>
<ul>
<li><a href="#model">model</a></li>
</ul>
</li>
<li><a href="#network-analysis">Network analysis</a></li>
<li><a href="#synapse-plasticity">synapse plasticity</a>
<ul>
<li><a href="#stdp">STDP</a></li>
<li><a href="#btsp">BTSP</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#neural-data-recording">neural data recording</a>
<ul>
<li><a href="#whole-brain-imaging">whole brain imaging</a>
<ul>
<li><a href="#drosophila-data">drosophila data</a></li>
<li><a href="#other-data">other data</a></li>
</ul>
</li>
<li><a href="#cell-tracking">cell tracking</a></li>
<li><a href="#wormid">wormid</a></li>
</ul>
</li>
<li><a href="#neural-data-analysis--modeling">neural data analysis &amp; modeling</a>
<ul>
<li><a href="#reivew">reivew</a></li>
<li><a href="#high-dimensional-data">high dimensional data</a></li>
<li><a href="#dimensional-reduction">dimensional reduction</a></li>
<li><a href="#manifold-reconstriction">manifold reconstriction</a></li>
<li><a href="#activity-analysis">activity analysis</a></li>
<li><a href="#head-direction">head direction</a></li>
</ul>
</li>
<li><a href="#encoding-and-decoding">encoding and decoding</a>
<ul>
<li><a href="#deep-learning-method">deep learning method</a>
<ul>
<li><a href="#fundation-model">fundation model</a></li>
<li><a href="#diffusion-model-visual-stimulus">diffusion model/ visual Stimulus</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#behavior">behavior</a>
<ul>
<li><a href="#oscilation">oscilation</a></li>
<li><a href="#place-cell-and-grid-cell">place cell and grid cell</a></li>
</ul>
</li>
<li><a href="#vision">vision</a>
<ul>
<li><a href="#drosophila">drosophila</a></li>
</ul>
</li>
<li><a href="#others">others</a>
<ul>
<li><a href="#diffusion-model">diffusion model</a></li>
<li><a href="#ml">ML</a></li>
<li><a href="#hippocamus">hippocamus</a></li>
<li><a href="#gene">gene</a></li>
<li><a href="#cognition">cognition</a>
<ul>
<li><a href="#attention">attention</a></li>
<li><a href="#learning-and-memory">learning and memory</a></li>
<li><a href="#decision-making">decision making</a></li>
</ul>
</li>
<li><a href="#ds--or">DS &amp; OR</a>
<ul>
<li><a href="#quadratic-assignment-problem">Quadratic assignment problem</a></li>
<li><a href="#similarity-matrix">similarity matrix</a></li>
</ul>
</li>
<li><a href="#ml-1">ML</a>
<ul>
<li><a href="#nas">NAS</a></li>
<li><a href="#cnn-kernel">CNN kernel</a></li>
</ul>
</li>
<li><a href="#sparse-decoding">sparse decoding</a></li>
<li><a href="#others-1">others</a></li>
<li><a href="#wiki">wiki</a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#todo">TODO</a></li>
</ul>
<hr>
<p>wiki of drosophila:
<a href="https://www.sdbonline.org/sites/fly/aimorph/brain3.htm">https://www.sdbonline.org/sites/fly/aimorph/brain3.htm</a>
The Interactive Fly</p>
<p>Genes involved in tissue and organ development</p>
<h2 id="connectome">Connectome</h2>
<ul>
<li>
<p><a href="https://codex.flywire.ai/academy_home#college">FlyWire for College Classrooms</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.06.08.544257v1">Not everything, not everywhere, not all at once: a study of brain-wide encoding of movement</a></p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/62362">Accelerating with FlyBrainLab the discovery of the functional logic of the Drosophila brain in the connectomic and synaptomic era</a></p>
</li>
<li>
<p><a href="https://elifesciences.org/reviewed-preprints/91532#tab-content">Deciphering the Genetic Code of Neuronal Type Connectivity: A Bilinear Modeling Approach</a>
bilinear + Mu Qiao</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41583-023-00756-z">How deep is the brain? The shallow brain hypothesis</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.10.04.616605v2">Generating brain-wide connectome using synthetic axonal morphologies</a>
Blue Brain Project</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41593-024-01812-2">The cell-type underpinnings of the human functional cortical connectome</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.02.10.528036v3">Evolution of connectivity architecture in the Drosophila mushroom body</a></p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/62576">The connectome of the adult Drosophila mushroom body provides insights into function</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41583-024-00879-x">Fly connectome over the wire</a>
<strong>Review, October 2024</strong></p>
</li>
</ul>
<h3 id="connectome-dataset">connectome dataset</h3>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-07558-y">Neuronal wiring diagram of an adult brain</a>
<strong>flywire, female</strong></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-07686-5">Whole-brain annotation and multi-connectome cell typing of Drosophila</a>  Nature 2024
FlyWire</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-07981-1">Neuronal parts list and wiring diagram for a visual system</a> FlyWire</p>
</li>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.add9330">The connectome of an insect brain</a>
<strong>larval brain</strong></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-07600-z">Synaptic architecture of leg and wing premotor control networks in Drosophila</a>  John C. Tuthill
<strong>motor</strong></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-07389-x">Connectomic reconstruction of a female Drosophila ventral nerve cord</a>
Wei-Chung Allen Lee &amp; John C. Tuthill</p>
</li>
<li>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(23)00719-5">Mapping of neuronal and glial primary cilia contactome and connectome in the human cerebral cortex</a>
human</p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663979v1">Connectomic reconstruction from hippocampal CA3 reveals spatially graded mossy fiber inputs and selective feedforward inhibition to pyramidal cells</a>
H. Sebastian Seung, David W. Tank</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-025-08746-0">Connectome-driven neural inventory of a complete visual system</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.04.16.589741v2">bioRxiv 2024</a>
Janelia, 2025, Nature,
a new connectome of the <strong>right optic lobe</strong> from a male Drosophila central nervous system FIB-SEM volume and a comprehensive inventory of the fly’s visual neurons</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-025-09276-5">Eye structure shapes neuron function in Drosophila motion vision</a>
Janelia, 2025, Nature,<br>
Davi D. Bock &amp; Michael B. Reiser</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0012160613001875">Waves of differentiation in the fly visual system</a>
Developmental Biology</p>
</li>
</ul>
<p><strong>MICrONS Project</strong></p>
<ul>
<li>
<p><a href="https://www.nature.com/collections/bdigiaicbd">Collection: The MICrONS Project</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-025-08790-w">Functional connectomics spanning multiple areas of mouse visual cortex</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-025-08840-3">Functional connectomics reveals general wiring rule in mouse visual cortex</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-07780-8">Inhibitory specificity from a connectomic census of mouse visual cortex</a></p>
</li>
<li>
<p><a href="https://www.cell.com/cell/pdf/S0092-8674(18)30787-6.pdf">A Complete Electron Microscopy Volume of the Brain of Adult Drosophila melanogaster</a></p>
</li>
<li>
<p><a href="https://www.cell.com/neuron/abstract/S0896-6273(25)00800-1">Integrative analysis of single-neuron projectomes links connectome, transcriptome, and function in the mouse cortex</a>
Xiaohong Xu, Chun Xu, Xiaofei Wang, Jun Yan, 2026, Neuron</p>
</li>
</ul>
<h3 id="technology-of-connectome">technology of connectome</h3>
<ul>
<li>
<p><a href="https://openreview.net/forum?id=lHDVYDy5S9l">Bridging the Gap: Point Clouds for Merging Neurons in Connectomics</a><br>
MIDL 2022</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41592-021-01183-7">Automatic detection of synaptic partners in a whole-brain Drosophila electron microscopy data set</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.11.04.467197v1.abstract">3D reconstruction of cell nuclei in a full Drosophila brain</a></p>
</li>
<li>
<p><a href="https://www.cell.com/cell/fulltext/S0092-8674(24)00307-6">Neurotransmitter classification from electron microscopy images at synaptic sites in Drosophila melanogaster</a> 2024
Machine learning identifies synaptic transmitters from electron micrographs</p>
</li>
</ul>
<h4 id="connection-reconsturction">connection reconsturction</h4>
<ul>
<li>
<p><a href="https://arxiv.org/abs/1210.1544">Reconstruction of Sparse Circuits Using Multi-neuronal Excitation (RESCUME)</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-025-08985-1">Light-microscopy-based connectomic reconstruction of mammalian brain tissue</a></p>
</li>
</ul>
<h3 id="connectome-analysis">connectome analysis</h3>
<ul>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2025.07.31.667571v2">Distributed control circuits across a brain-and-cord connectome</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-07968-y">Network statistics of the whole-brain connectome of Drosophila</a>
<strong>Mala Murthy, Nature</strong></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.06.27.546055v2">Whole-brain annotation and multi-connectome cell typing quantifies circuit stereotypy in Drosophila</a>
<strong>hemibrain and flywire</strong></p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/57443">A connectome and analysis of the adult Drosophila central brain</a> <strong>hemibrain</strong></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.05.01.592061v1">Brain rewiring during developmental transitions: A Comparative Analysis of <strong>Larva and Adult</strong> Drosophila melanogaster</a></p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/79139">Connectomic analysis of the Drosophila lateral neuron clock cells reveals the synaptic basis of functional pacemaker classes</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1053811920308326">Cross-species functional alignment reveals evolutionary hierarchy within the connectome</a></p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/66039">A connectome of the Drosophila central complex reveals network motifs suitable for flexible navigation and context-dependent action selection</a></p>
</li>
<li>
<p><a href="https://www.eneuro.org/content/9/2/ENEURO.0053-22.2022">Identifying Inputs to Visual Projection Neurons in Drosophila Lobula by Analyzing Connectomic Data</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41593-024-01640-4">Hue selectivity from recurrent circuitry in Drosophila</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0925231220317926">Multilayer network analysis of <strong>C. elegans</strong>: Looking into the locomotory circuitry</a></p>
</li>
<li>
<p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005283">The Multilayer Connectome of Caenorhabditis <strong>elegans</strong></a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.11.30.569343v1.abstract">Exploring the Brain Characteristics of Structure-informed Functional Connectivity through Graph Attention Network</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-07982-0">The fly connectome reveals a path to the <strong>effectome</strong></a><br>
Gregory S. X. E. Jefferis, Mala Murthy &amp; Jonathan W. Pillow
From connectome to effectome: learning the causal interaction map of the fly brain</p>
</li>
<li>
<p><a href="https://www.jneurosci.org/content/43/37/6384">Hierarchical Modular Structure of the Drosophila Connectome</a>
We defined analogous measures for cell types. To identify cell types that may be partitioned into dense subnetworks, we introduce the Fraction of Type measure,</p>
</li>
</ul>
<p class="katex-block"><span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mi>F</mi><mi>T</mi><mo stretchy="false">(</mo><mi>T</mi><mo separator="true">;</mo><mi>X</mi><mo stretchy="false">)</mo><mo>=</mo><munder><mo>∑</mo><mrow><mi>c</mi><mo>∈</mo><mi>C</mi><mo stretchy="false">(</mo><mi>X</mi><mo stretchy="false">)</mo></mrow></munder><mo stretchy="false">(</mo><mfrac><msub><mi>n</mi><mrow><mi>c</mi><mo>&amp;</mo><mi>T</mi></mrow></msub><msub><mi>n</mi><mi>T</mi></msub></mfrac><msup><mo stretchy="false">)</mo><mn>2</mn></msup></mrow><annotation encoding="application/x-tex">FT(T;X) = \sum_{c \in C(X)} (\frac{n_{c \And T}}{n_T})^2
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord mathnormal" style="margin-right:0.13889em;">FT</span><span class="mopen">(</span><span class="mord mathnormal" style="margin-right:0.13889em;">T</span><span class="mpunct">;</span><span class="mspace" style="margin-right:0.1667em;"></span><span class="mord mathnormal" style="margin-right:0.07847em;">X</span><span class="mclose">)</span><span class="mspace" style="margin-right:0.2778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em;"></span></span><span class="base"><span class="strut" style="height:2.6236em;vertical-align:-1.516em;"></span><span class="mop op-limits"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.05em;"><span style="top:-1.809em;margin-left:0em;"><span class="pstrut" style="height:3.05em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathnormal mtight">c</span><span class="mrel mtight">∈</span><span class="mord mathnormal mtight" style="margin-right:0.07153em;">C</span><span class="mopen mtight">(</span><span class="mord mathnormal mtight" style="margin-right:0.07847em;">X</span><span class="mclose mtight">)</span></span></span></span><span style="top:-3.05em;"><span class="pstrut" style="height:3.05em;"></span><span><span class="mop op-symbol large-op">∑</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:1.516em;"><span></span></span></span></span></span><span class="mopen">(</span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.1076em;"><span style="top:-2.314em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord"><span class="mord mathnormal">n</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.3283em;"><span style="top:-2.55em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathnormal mtight" style="margin-right:0.13889em;">T</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span><span style="top:-3.23em;"><span class="pstrut" style="height:3em;"></span><span class="frac-line" style="border-bottom-width:0.04em;"></span></span><span style="top:-3.677em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord"><span class="mord mathnormal">n</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.3361em;"><span style="top:-2.55em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathnormal mtight">c</span><span class="mbin mtight">&amp;</span><span class="mord mathnormal mtight" style="margin-right:0.13889em;">T</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.836em;"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span><span class="mclose"><span class="mclose">)</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.8641em;"><span style="top:-3.113em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span></span></span></span></span></span></span></span></span></p>
<ul>
<li>
<p><a href="https://codex.flywire.ai/app/visual_columns_map?dataset=fafb">Visual Columns Map</a>
A comprehensive mapping of the 31 columnar cell types in the compound eye has been achieved using a computational optimization technique based on synaptic connectivity. A manuscript describing the resource and methodology is currently in preparation. The quality of this mapping can be evaluated through specially developed visualizations available below.</p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679410v2.full.pdf">The Connectome Interpreter Toolkit</a>
<a href="https://www.biorxiv.org/content/10.1101/2025.09.29.679410v2">https://www.biorxiv.org/content/10.1101/2025.09.29.679410v2</a></p>
</li>
</ul>
<h3 id="gap-junction">gap junction</h3>
<ul>
<li><a href="https://www.cell.com/article/S1074-5521(05)00072-4/fulltext">Intercellular Communication: the Drosophila Innexin Multiprotein Family of Gap Junction Proteins</a>
Review, 2005</li>
</ul>
<h3 id="connectome-and-gene">connectome and gene</h3>
<p><strong>review</strong></p>
<ul>
<li>
<p><a href="https://rupress.org/jcb/article/220/7/e202103052/212258/The-cell-biology-of-synapse-formationMechanisms-of">The cell biology of synapse formation</a> <strong>SAM</strong><br>
Review, 2021, from <a href="https://med.stanford.edu/sudhoflab.html">Thomas C. Südhof</a><br>
![alt text](figs/synaptic adhesion molecules.png)
synaptic adhesion molecules.png</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627318308420">Towards an Understanding of Synapse Formation</a>
Review, 2018, from Thomas C. Südhof</p>
</li>
<li>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(13)00916-1">The Dynamic Synapse</a> Review, 2013, Neuron, Daniel Choquet, Antoine Triller</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627302008760">Making Connections in the Fly Visual System</a>
Review, 2002, <a href="https://flyvisionlab.weebly.com/">Thomas R Clandinin</a>, S.Lawrence Zipursky</p>
</li>
<li>
<p><a href="https://www.cell.com/fulltext/S0896-6273(04)00498-2">Synaptic Connectivity and Neuronal Morphology: Two Sides of the Same Coin</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867415011939">Beyond Molecular Codes: Simple Rules to Wire Complex Brains</a>
Review, 2015, Bassem A. Hassan, P. Robin Hiesinger</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S095943882500087X">Principles of synaptogenesis: Insights from Caenorhabditis elegans</a>
Elisa B. Frankel, Peri T. Kurshan, 2025, Current Opinion in Neurobiology</p>
</li>
<li>
<p><a href="https://www.cell.com/fulltext/S0092-8674(10)01145-1">Chemoaffinity Revisited: Dscams, Protocadherins, and Neural Circuit Assembly</a>
Teview, 2010, S. Lawrence Zipursky, Joshua R. Sanes</p>
</li>
<li>
<p><a href="https://www.science.org/doi/full/10.1126/science.abg7285">Architectures of neuronal circuits</a>
reivew, 2021, Science, Liqun Luo
<img src="https://www.science.org/cms/10.1126/science.abg7285/asset/59d561d9-7d95-456d-97e5-1b811c05023f/assets/images/large/science.abg7285-f5.jpg" alt="alt text"></p>
</li>
<li>
<p><a href="https://www.cell.com/trends/neurosciences/fulltext/S0166-2236(25)00238-3">Reconstructing the <strong>human</strong> brain’s wiring diagram from axons up</a><br>
Review, 2026</p>
</li>
</ul>
<p><strong>computational methods</strong></p>
<ul>
<li>
<p><strong>bilinear:</strong> <a href="https://www.pnas.org/doi/full/10.1073/pnas.2009093117">Uncovering the genetic blueprint of the C. elegans nervous system</a>
István A. Kovács, Dániel L. <strong>Barabási</strong>, and Albert-László <strong>Barabási</strong>  PNAS</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0306457324002917">Graph similarity learning for cross-level interactions</a>
“Incorporates direction attributes via a bilinear model for better adaptability to data.”</p>
</li>
<li>
<p><strong>biclique motifs</strong> <a href="https://www.sciencedirect.com/science/article/pii/S0896627319309262">A Genetic Model of the Connectome</a>
Dániel L. Barabási, Albert-László Barabási</p>
</li>
<li>
<p><strong>bilinear</strong> <a href="https://elifesciences.org/reviewed-preprints/91532">Deciphering the Genetic Code of Neuronal Type Connectivity: A Bilinear Modeling Approach</a>
<strong>Mu Qiao</strong></p>
</li>
<li>
<p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010382"><strong>Constructive connectomics</strong>: How neuronal axons get from here to there using gene- expression maps derived from their family trees</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.07.01.601562v1">Building a small brain with a simple <strong>stochastic generative model</strong></a>
Oren Richter, Elad Schneidman</p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.03.15.532611v1">The structure and function of neural connectomes are shaped by a small number of design principles</a>
Elad Schneidman</p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2022.11.01.514680v1.abstract">Cell lineage predicts neural connectivity beyond cell type</a> C. elegans<br>
Konrad P. Kording,</p>
</li>
<li>
<p><a href="https://www.frontiersin.org/journals/computational-neuroscience/articles/10.3389/neuro.10.025.2009/full">A framework for modeling the growth and development of neurons and networks</a>
Frederic Zubler and Rodney Douglas(ETH Zurich), 2009</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41467-025-68059-8">Uncovering the molecular logic of cortical wiring between neuronal subtypes across development through ligand–receptor inference</a>
Ludovic Telley &amp; Antoine de Chevigny, 2026, Nature Communications</p>
</li>
</ul>
<p><strong>experimental test</strong></p>
<ul>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2025.03.01.640986v1">Rewiring an olfactory circuit by altering the combinatorial code of cell-surface proteins</a>
<strong>application</strong> Liqun Luo</p>
</li>
<li>
<p><a href="https://www.cell.com/developmental-cell/fulltext/S1534-5807(24)00638-5">Biased cell adhesion organizes the Drosophila visual motion integration circuit</a></p>
</li>
<li>
<p>Yannick Carrier, .., Filipe Pinto-Teixeira, 2025, Developmental Cell
Beat/Side interactions are not required for synapse formation<br>
Beat/Side interactions contribute to synaptic specificity<br>
Beat/Side interactions bias cellular adjacency prior to synaptogenesis</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0960982223010631">Brain wiring determinants uncovered by integrating connectomes and transcriptomes</a>
<strong>Yoo2023</strong> T4_T5</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867421007583">Molecular topography of an entire nervous system</a>
<strong>key_and_lock</strong></p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/64903#metrics">The Prop1-like homeobox gene unc-42 specifies the identity of synaptically connected neurons</a>
<strong>C. elegans</strong></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627319300091">Cell-type-Specific Patterned Stimulus-Independent Neuronal Activity in the Drosophila Visual System during Synapse Formation</a>
<strong>S. Lawrence Zipursky</strong></p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/19942">Genome-wide identification of neuronal activity-regulated genes in Drosophila</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.12.14.628490v1">Dissecting origins of wiring specificity in dense cortical connectomes</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41380-022-01519-5">Connectome gradient dysfunction in major depression and its association with gene expression profiles and treatment outcomes</a></p>
</li>
</ul>
<p>data:</p>
<ul>
<li><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005989">Hub connectivity, neuronal diversity, and gene expression in the Caenorhabditis elegans connectome</a></li>
</ul>
<p><strong>other spicies:</strong></p>
<ul>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.adv2609">Constrained roads to complex brains</a>
perspective, Neural development and brain circuit evolution converged in birds and <strong>mammals</strong></p>
</li>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.adp3411">Evolutionary convergence of sensory circuits in the pallium of amniotes</a></p>
</li>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.adp3957">Enhancer-driven cell type comparison reveals similarities between the mammalian and bird pallium</a></p>
</li>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.adp5182">Developmental origins and evolution of pallial cell types and structures in birds</a></p>
</li>
</ul>
<h4 id="rl-and-logical-rule">RL and logical rule</h4>
<ul>
<li>
<p><a href="https://arxiv.org/abs/1702.08367">Differentiable Learning of Logical Rules for Knowledge Base Reasoning</a>
2017</p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2206.01483">Finding Rule-Interpretable Non-Negative Data Representation</a>
Matej Mihelčić, Pauli Miettinen</p>
</li>
</ul>
<h4 id="gene-data-of-drosophila">gene data of Drosophila</h4>
<p><a href="gene_data.html">gene data of Drosophila</a></p>
<h4 id="other-methods-to-reconstruct-connectome">other methods to reconstruct connectome</h4>
<ul>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.07.01.601562v1">Building a small brain with a simple stochastic generative model</a>
Oren Richter, Elad Schneidman</p>
</li>
<li>
<p><a href="https://www.cell.com/cell/fulltext/S0092-8674(24)00307-6">Neurotransmitter classification from electron microscopy images at synaptic sites in Drosophila melanogaster</a>
Six transmitters predicted across the whole fly brain connectome:
<a href="https://www.cell.com/cms/10.1016/j.cell.2024.03.016/asset/24723725-eb0b-40f7-b29d-10f859442e2b/main.assets/gr5_lrg.jpg">https://www.cell.com/cms/10.1016/j.cell.2024.03.016/asset/24723725-eb0b-40f7-b29d-10f859442e2b/main.assets/gr5_lrg.jpg</a>
Schematic of a neuron broken into its neuronal compartments. Inset, the proportion of presynapses in each of the four compartment types.</p>
</li>
</ul>
<h4 id="human-connectome">human connectome</h4>
<ul>
<li>
<p><a href="https://www.jneurosci.org/content/45/7/e1510242024">Topographic Axes of Wiring Space Converge to Genetic Topography in Shaping the Human Cortical Layout</a>
<strong>3GCs(global connectopies)</strong></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1053811919300114">A practical guide to linking brain-wide gene expression and neuroimaging data</a>
seven major steps to link expression measures and neuroimaging data</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/nrn3586">Molecular logic of neocortical projection neuron specification, development and diversity</a>, Neocortical projection neurons</p>
</li>
</ul>
<!-- ![alt text|200pt](<figs/Neocortical projection neurons.png>) -->
<p><img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fnrn3586/MediaObjects/41583_2013_Article_BFnrn3586_Fig1_HTML.jpg?as=webp" alt="alt text|200pt"></p>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2024.05.07.593018v1"><strong>Benchmarking methods for mapping functional connectivity in the brain</strong></a>
Here we use a library of <strong>239 pairwise statistics</strong> to benchmark canonical features of <strong>FC networks</strong>, including hub mapping, weight–distance trade-offs, structure–function coupling, correspondence with other neurophysiological networks, individual fingerprinting and brain–behavior prediction.</li>
</ul>
<h4 id="experiment-support">experiment support</h4>
<ul>
<li><a href="https://www.sciencedirect.com/science/article/pii/S2211124724001268">Complex formation of immunoglobulin superfamily molecules Side-IV and Beat-IIb regulates synaptic specificity</a><br>
2023, Cell Reports
<ul>
<li>Side-IV binding to Beat-IIb regulates synaptic specificity in vivo</li>
<li>Side-IV/Beat-IIb forms complex with Kirre and Dsyd-1</li>
</ul>
</li>
</ul>
<p><a href="https://www.nature.com/articles/s41586-025-09768-4">Repulsions instruct synaptic partner matching in an olfactory circuit</a>
Zhuoran Li, Cheng Lyu, ..., Engin Özkan &amp; Liqun Luo, 2026, Nature</p>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s41467-020-15981-8">DIP/Dpr interactions and the evolutionary design of specificity in protein families</a>
binding specificity of <strong>DIP/Dpr</strong> subgroups is controlled by &quot;negative constraints&quot;, which interfere with binding</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0960982205014715">A Conserved Role for Drosophila Neuroglian and Human L1-CAM in Central-Synapse Formation</a><br>
2006, Current Biology</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1534580724001485?via%3Dihub">Spatial patterning controls neuron numbers in the Drosophila visual system</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-07221-6">Whole-cortex in situ sequencing reveals input-dependent area identity</a>
Zador lab</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627319300091">Cell-type-Specific Patterned Stimulus-Independent Neuronal Activity in the Drosophila Visual System during Synapse Formation</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0959437X07000937">Pattern formation in the Drosophila eye</a><br>
2007, Review, 2007, Richard W Carthew, Northwestern University</p>
</li>
<li>
<p><a href="https://journals.biologists.com/dev/article/132/10/2251/42750/Novel-brain-wiring-functions-for-classical">Novel brain wiring functions for classical morphogens: a role as graded positional cues in axon guidance</a><br>
morphogen gradients also serve to guide axonal pathfinding during development of the nervous system</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867415015019">Ig Superfamily Ligand and Receptor Pairs Expressed in Synaptic Partners in Drosophila</a>
S. Lawrence Zipursky 2015, Cell</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41593-024-01857-3">The developmental emergence of reliable cortical representations</a>
we examined this ‘nature–nurture transform’ at the single-trial level using chronic in vivo calcium imaging in <strong>ferret</strong> visual cortex.</p>
</li>
</ul>
<h4 id="geneweave-application">GeneWeave application</h4>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2024.08.07.606541v1">Stochastic Wiring of Cell Types Enhances Fitness by Generating Phenotypic Variability</a>
Divyansha Lachi, Ann Huang, Augustine N. Mavor-Parker, Arna Ghosh, Blake Richards, Anthony Zador</li>
</ul>
<p>connection probabilities between discrete cell types are genetically specified, to investigate
the benefits of stochasticity in the development of neural wiring</p>
<h3 id="cell-cell-communication-and-interaction">cell-cell communication and interaction</h3>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s41596-020-0292-x">CellPhoneDB: inferring <strong>cell–cell communication</strong> from combined expression of multi-subunit ligand–receptor complexes</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41576-020-00292-x">Deciphering cell–cell interactions and communication from gene expression</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41467-024-51329-2">Deciphering cell–cell communication at single-cell resolution for spatial transcriptomics with subgraph-based graph attention network</a></p>
</li>
<li>
<p><a href="https://vcg.seas.harvard.edu/publications/cell2cell">Cell2Cell: Explorative Cell Interaction Analysis in Multi-Volumetric Tissue Data</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41467-021-21246-9">Inference and analysis of cell-cell communication using CellChat</a></p>
</li>
</ul>
<h3 id="protein-interaction">protein interaction</h3>
<ul>
<li><a href="https://paperswithcode.com/dataset/ppi">PPI (Protein-Protein Interactions (PPI))</a></li>
</ul>
<h4 id="experiment-support-1">experiment support</h4>
<ul>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.07.08.602367v2">Nematode Extracellular Protein Interactome Expands Connections between Signaling Pathways</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867413007083">An Extracellular Interactome of Immunoglobulin and LRR Proteins Reveals Receptor-Ligand Networks</a>
cell, 2013
<img src="https://ars.els-cdn.com/content/image/1-s2.0-S0092867413007083-gr3_lrg.jpg" alt="alt text|200pt"></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41597-024-03233-z">A comprehensive dataset of protein-protein interactions and ligand binding pockets for advancing drug discovery</a>
<strong>PPIs</strong></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1806.04398">Attentive cross-modal paratope prediction</a>
Antibodies</p>
</li>
<li>
<p><a href="https://journals.biologists.com/dev/article/139/18/3442/45247/Principles-of-branch-dynamics-governing-shape">Principles of branch dynamics governing shape characteristics of cerebellar Purkinje cell dendrites</a>
Growing dendrites are retracted or stalled by contacts with other dendrites.</p>
</li>
<li>
<p><a href="https://www.jneurosci.org/content/19/17/7548">Neuropil Pattern Formation and Regulation of Cell Adhesion Molecules in Drosophila Optic Lobe Development Depend on Synaptobrevin</a>
1999</p>
</li>
</ul>
<h4 id="deep-learning-methods">deep learning methods</h4>
<ul>
<li>
<p>paper with code: <a href="https://paperswithcode.com/dataset/ppi">https://paperswithcode.com/dataset/ppi</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/d41586-020-00018-3">Powerful antibiotics discovered using AI</a></p>
</li>
<li>
<p><a href="https://link.springer.com/article/10.1186/s12859-022-04910-9">Struct2Graph: a graph attention network for structure based predictions of protein–protein interactions</a></p>
</li>
<li>
<p><a href="https://academic.oup.com/bib/article/23/1/bbab502/6457163">Accurate protein function prediction via graph attention networks with predicted structure information</a></p>
</li>
</ul>
<h4 id="result-verification">result verification</h4>
<ul>
<li><a href="https://www.cell.com/cell/fulltext/S0092-8674(24)01093-6">A conserved fertilization complex bridges sperm and egg in vertebrates</a> <strong>AlphaFold-Multimer</strong>
This approach employs structural predictions of pairwise protein-protein interactions between each bait protein and the candidate proteins. A high interface-predicted template modeling score (<strong>ipTM score</strong>) reflects high confidence in the relative positioning of the interfaces.</li>
</ul>
<!-- ![alt text|200pt](<figs/sperm and egg.png>) -->
<p><img src="https://www.cell.com/cms/10.1016/j.cell.2024.09.035/asset/94c10a25-098f-4905-a883-7b118075f5de/main.assets/fx1_lrg.jpg" alt="alt text|200pt"></p>
<h3 id="data-mapping-between-transcriptome-and-connectome">data mapping between transcriptome and connectome</h3>
<ul>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S009286742031254X">Integrated Morphoelectric and Transcriptomic Classification of Cortical GABAergic Cells</a>
Cell2020, <strong>all kinds of neurons!!!</strong></p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/13222">Bicoid gradient formation and function in the Drosophila pre-syncytial blastoderm</a></p>
</li>
<li>
<p><a href="https://genesdev.cshlp.org/content/11/15/1963">An anteroposterior Dorsal gradient in theDrosophila embryo</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0960982213003333">Variation in the Dorsal Gradient Distribution Is a Source for Modified Scaling of Germ Layers in Drosophila</a></p>
</li>
<li>
<p><a href="https://www.nature.com/scitable/topicpage/gradient-based-dna-transcription-control-in-animals-1062/">Gradient-Based DNA Transcription Control in Animals</a></p>
</li>
<li>
<p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011574">Inferring causal connectivity from pairwise recordings and optogenetics</a>
Konrad P. Kording</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41593-024-01806-0">Opportunities and challenges of single-cell and spatially resolved genomics methods for neuroscience discovery</a></p>
</li>
</ul>
<h4 id="neural-type">neural type</h4>
<p><a href="../connectivity_to_type.html">my blog on connectivity_to_type</a></p>
<ul>
<li>
<p><a href="https://elifesciences.org/articles/04250">Automatic discovery of cell types and microcircuitry from neural connectomics</a><br>
Eric Jonas, Konrad Kording<br>
It combines the information traditionally used by biologists in a principled and probabilistically coherent manner, including connectivity, cell body location, and the spatial distribution of synapses.</p>
</li>
<li>
<p><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)01134-9">Evidence from spatial transcriptomics for the mosaic hypothesis and pure cell types in the cortex</a>
Uygar Sümbül
Allen</p>
</li>
</ul>
<p>Spatial transcriptomic data provide evidence for the mosaic hypothesis in the mouse cortex<br>
Somata of neurons of the same type appear to avoid spatial proximity with each other</p>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s41592-024-01806-0">Connectivity of single neurons classifies cell subtypes in mouse brains</a>
Neuronal Connectivity as a Determinant of Cell Types and Subtypes</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867416300034">Bayesian Sparse Regression Analysis Documents the Diversity of Spinal Inhibitory Interneurons</a><br>
<a href="https://ars.els-cdn.com/content/image/1-s2.0-S0092867416300034-mmc2.pdf">SI</a> <strong>Bayesian method</strong></p>
</li>
</ul>
<p class="katex-block"><span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mi>p</mi><mo stretchy="false">(</mo><mi mathvariant="bold">y</mi><mo>∣</mo><mi>Q</mi><mo separator="true">,</mo><mi mathvariant="bold">f</mi><mo stretchy="false">)</mo><mo>∝</mo><msup><mi>e</mi><mrow><mo>−</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><msup><mi mathvariant="bold">f</mi><mo mathvariant="normal" lspace="0em" rspace="0em">′</mo></msup><mrow><mi mathvariant="bold">M</mi><mi mathvariant="bold">f</mi></mrow><mo>+</mo><msup><mi mathvariant="bold">y</mi><mo mathvariant="normal" lspace="0em" rspace="0em">′</mo></msup><mi>Q</mi><mi mathvariant="bold">f</mi></mrow></msup></mrow><annotation encoding="application/x-tex">p(\mathbf{y} \mid Q, \mathbf{f}) \propto e^{-\frac{1}{2} \mathbf{f}^{\prime} \mathbf{M f}+\mathbf{y}^{\prime} Q \mathbf{f}}
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord mathnormal">p</span><span class="mopen">(</span><span class="mord mathbf" style="margin-right:0.01597em;">y</span><span class="mspace" style="margin-right:0.2778em;"></span><span class="mrel">∣</span><span class="mspace" style="margin-right:0.2778em;"></span></span><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord mathnormal">Q</span><span class="mpunct">,</span><span class="mspace" style="margin-right:0.1667em;"></span><span class="mord mathbf" style="margin-right:0.10903em;">f</span><span class="mclose">)</span><span class="mspace" style="margin-right:0.2778em;"></span><span class="mrel">∝</span><span class="mspace" style="margin-right:0.2778em;"></span></span><span class="base"><span class="strut" style="height:1.004em;"></span><span class="mord"><span class="mord mathnormal">e</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:1.004em;"><span style="top:-3.413em;margin-right:0.05em;"><span class="pstrut" style="height:3em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">−</span><span class="mord mtight"><span class="mopen nulldelimiter sizing reset-size3 size6"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8443em;"><span style="top:-2.656em;"><span class="pstrut" style="height:3em;"></span><span class="sizing reset-size3 size1 mtight"><span class="mord mtight"><span class="mord mtight">2</span></span></span></span><span style="top:-3.2255em;"><span class="pstrut" style="height:3em;"></span><span class="frac-line mtight" style="border-bottom-width:0.049em;"></span></span><span style="top:-3.384em;"><span class="pstrut" style="height:3em;"></span><span class="sizing reset-size3 size1 mtight"><span class="mord mtight"><span class="mord mtight">1</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.344em;"><span></span></span></span></span></span><span class="mclose nulldelimiter sizing reset-size3 size6"></span></span><span class="mord mtight"><span class="mord mathbf mtight" style="margin-right:0.10903em;">f</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.8278em;"><span style="top:-2.931em;margin-right:0.0714em;"><span class="pstrut" style="height:2.5em;"></span><span class="sizing reset-size3 size1 mtight"><span class="mord mtight"><span class="mord mtight">′</span></span></span></span></span></span></span></span></span><span class="mord mtight"><span class="mord mathbf mtight" style="margin-right:0.10903em;">Mf</span></span><span class="mbin mtight">+</span><span class="mord mtight"><span class="mord mathbf mtight" style="margin-right:0.01597em;">y</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.8278em;"><span style="top:-2.931em;margin-right:0.0714em;"><span class="pstrut" style="height:2.5em;"></span><span class="sizing reset-size3 size1 mtight"><span class="mord mtight"><span class="mord mtight">′</span></span></span></span></span></span></span></span></span><span class="mord mathnormal mtight">Q</span><span class="mord mathbf mtight" style="margin-right:0.10903em;">f</span></span></span></span></span></span></span></span></span></span></span></span></span></p>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s41586-023-06808-9">Molecularly defined and spatially resolved cell atlas of the whole mouse brain</a>
Xiaowei Zhuang</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-021-03941-1">Morphological diversity of single neurons in molecularly defined cell types</a> 2021 Nature, Hongkui Zeng, Allen Institute</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41592-019-0631-4">Probabilistic cell typing enables fine mapping of closely related cell types in situ</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S2211124722009895">Single-neuron models linking electrophysiology, morphology, and transcriptomics across cortical cell types</a>
9,200 models are generated that capture multimodal single-cell datasets</p>
</li>
</ul>
<h4 id="neural-activity-and-connectivity">neural activity and connectivity</h4>
<ul>
<li>
<p><a href="https://arxiv.org/pdf/2411.12091">Homogenized C. elegans Neural Activity and Connectivity Data</a></p>
</li>
<li>
<p><a href="https://arxiv.org/pdf/2411.12091">Homogenized C. elegans Neural Activity and Connectivity Data</a>
<a href="https://arxiv.org/abs/2411.12091">https://arxiv.org/abs/2411.12091</a>
Konrad P Kording, Edward S Boyden</p>
</li>
</ul>
<h3 id="morphology">Morphology</h3>
<ul>
<li>
<p><a href="https://journals.physiology.org/doi/full/10.1152/jn.00865.2006">Mapping Function Onto Neuronal Morphology</a>
Klaus M. Stiefel, Terrence J. Sejnowski<br>
We used a Lindenmayer-system (L-system) (Lindenmayer 1968) for the algorithmic construction of model neuron morphologies. Simulations of electrophysiological dynamics were carried out in the neuronal simulation language NEURON (version 5.7) (Hines and Carnevale 1997, 2000).</p>
</li>
<li>
<p><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(23)00553-5">Neuronal contact predicts connectivity in the C. elegans brain</a><br>
Peters’ rule explains synaptic specificity throughout the C. elegans nervous system</p>
</li>
<li>
<p><a href="https://www.annualreviews.org/content/journals/10.1146/annurev-cellbio-100913-013020">Development of Dendritic Form and Function</a><br>
Review paper, 2015, Julie L. Lefebvre, Joshua R. Sanes, and Jeremy N. Kay</p>
</li>
<li>
<p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000877">One Rule to Grow Them All: A General Theory of Neuronal Branching and Its Practical Application</a><br>
Hermann Cuntz ,Friedrich Forstner,Alexander Borst,Michael Häusser, 2010</p>
</li>
<li>
<p><a href="https://www.frontiersin.org/journals/cellular-neuroscience/articles/10.3389/fncel.2012.00061/full">Molecular mechanisms of dendrite morphogenesis</a>
Review, 2012, Jyothi Arikkath</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0165027010006758">Automated analysis of neuronal morphology, synapse number and synaptic recruitment</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/nprot.2008.51">L-Measure: a web-accessible tool for the analysis, comparison and search of digital reconstructions of neuronal morphologies</a></p>
</li>
<li>
<p><a href="https://openreview.net/forum?id=p6hArCtwLAU">TreeMoCo: Contrastive Neuron Morphology Representation Learning</a></p>
</li>
<li>
<p><a href="https://journals.biologists.com/dev/article/139/18/3442/45247/Principles-of-branch-dynamics-governing-shape">Principles of branch dynamics governing shape characteristics of cerebellar Purkinje cell dendrites</a>
Growing dendrites are retracted or stalled by contacts with other dendrites.</p>
</li>
</ul>
<h4 id="morphology-data">morphology data</h4>
<p><a href="https://neuromorpho.org/">NeuroMorpho.Org</a> is a centrally curated inventorys of digitally reconstructed neurons and glia</p>
<h4 id="maximum-weighted-independent-set-mwis">maximum weighted independent set (MWIS)</h4>
<ul>
<li>
<p><a href="https://arxiv.org/abs/1810.10834">Exactly Solving the Maximum Weight Independent Set Problem on Large Real-World Graphs</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0166218X02002056">A note on greedy algorithms for the maximum weighted independent set problem</a></p>
</li>
</ul>
<p>Graph compression or sparsification</p>
<ul>
<li><a href="https://arxiv.org/pdf/2007.07862.pdf">Vertex Sparsification for Edge Connectivity</a></li>
</ul>
<h4 id="maximum-dispersion">Maximum Dispersion</h4>
<p>Maximum Diversity Problem (MDP)</p>
<ul>
<li>
<p><a href="https://arxiv.org/abs/1706.09185">Dispersion on Trees</a></p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/document/7004250">Estimating Pairwise Distances in Large Graphs</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0196677400911453">Approximation Algorithms for Dispersion Problems</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0167637797000345">Approximation Algorithms for Maximum Dispersion</a></p>
</li>
<li>
<p><a href="https://epubs.siam.org/doi/10.1137/0404048">Obnoxious Facility Location on Graphs</a></p>
</li>
<li>
<p><a href="https://zhuanlan.zhihu.com/p/101724381">变邻域搜索(VNS)算法求解Max-Mean Dispersion Problem</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1503.00980">On memetic search for the max-mean dispersion problem</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0950705115001860">Tabu search for the Max–Mean Dispersion Problem</a></p>
</li>
</ul>
<!-- - [Exact Solution of P-dispersion Problems](https://www.semanticscholar.org/paper/Exact-Solution-of-P-dispersion-Problems-Pisinger/1eb3810077c0af9d46ed5ff2b0819d954c97dcae) -->
<ul>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0305054804002552">Upper bounds and exact algorithms for p-dispersion problems
</a>
<a href="http://yalma.fime.uanl.mx/~roger/work/teaching/class_tso/docs_project/problems/PDP/cor-2006-Pisinger.pdf">http://yalma.fime.uanl.mx/~roger/work/teaching/class_tso/docs_project/problems/PDP/cor-2006-Pisinger.pdf</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0377221721006548">A review on discrete diversity and dispersion maximization from an OR perspective</a></p>
</li>
<li>
<p><a href="https://pubsonline.informs.org/doi/abs/10.1287/opre.42.2.299">Heuristic and special case algorithms for dispersion problems</a></p>
</li>
<li>
<p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/net.3230120206">A network flow solution to some nonlinear 0-1 programming problems, with applications to graph theory</a></p>
</li>
</ul>
<h3 id="connectome-visualization">connectome visualization</h3>
<ul>
<li>
<p><a href="https://flywire.ai/apps">FlyWire visualization tools</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.07.26.550598v1">CAVE: Connectome Annotation Versioning Engine</a>
<a href="https://vcg.seas.harvard.edu/publications/cave">https://vcg.seas.harvard.edu/publications/cave</a></p>
</li>
<li>
<p><a href="https://direct.mit.edu/netn/article/2/3/344/5434/NeuroCave-A-web-based-immersive-visualization">NeuroCave: A web-based immersive visualization platform for exploring connectome datasets</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.10.05.561103v1">SmartEM: machine-learning guided electron microscopy</a></p>
</li>
<li>
<p>BOOK: <a href="https://link.springer.com/chapter/10.1007/978-1-4471-6497-5_21">Visualization in Connectomics</a></p>
</li>
<li>
<p><a href="https://www.pnas.org/doi/full/10.1073/pnas.1103583108">A Unified attentional bottleneck in the human brain</a>
connection bottleneck</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41598-022-07959-x">Computation and visualization of cell–cell signaling topologies in single-cell systems data using Connectome</a></p>
</li>
<li>
<p>ViMO <a href="https://ieeexplore.ieee.org/document/10298029"> - Visual Analysis of Neuronal Connectivity Motifs</a></p>
</li>
<li>
<p><a href="https://networkrepository.com/">Network Repository</a>. An Interactive Scientific Network Data Repository.
the first scientific network data repository with interactive visual analytics.
new GraphVis: interactive visual graph mining and machine learning</p>
</li>
</ul>
<p><strong>human connectome</strong></p>
<ul>
<li>
<p><a href="https://doi.org/10.1162/netn_a_00044">NeuroCave: A web-based immersive visualization platform for exploring connectome datasets</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1908.02831">Visualizing the PHATE of Neural Networks</a><br>
<a href="https://direct.mit.edu/view-large/figure/1188158/00044fx01.tif">Table 1. A survey of neuroimaging connectomic software. </a></p>
</li>
</ul>
<h3 id="gene-data--transcriptome">gene data / transcriptome</h3>
<ul>
<li><a href="https://www.nature.com/articles/s41593-024-01806-0">Opportunities and challenges of single-cell and spatially resolved genomics methods for neuroscience discovery</a><br>
Nature Neuroscience volume 27, pages 2292-2309 (2024)<br>
Review</li>
</ul>
<h4 id="gene-data-for-drosophila">gene data for Drosophila</h4>
<ul>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S2666979X25003593">A high-resolution atlas of the brain predicts lineage and birth order underlying neuronal identity</a>
Cell Genomics, Dec 2025
Stephen F. Goodwin</p>
</li>
<li>
<p><a href="https://www.science.org/doi/abs/10.1126/science.abk2432">Fly Cell Atlas: A single-nucleus transcriptomic atlas of the adult fruit fly</a><br>
2022, science</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867418307207">A Single-Cell Transcriptome Atlas of the Aging Drosophila Brain</a>
Cell, 2018</p>
</li>
</ul>
<!-- - [A Single-Cell Transcriptome Atlas of the Aging Drosophila Brain](https://www.cell.com/cell/fulltext/S0092-8674(18)30720-7) -->
<ul>
<li>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(20)30774-1">Transcriptional Programs of Circuit Assembly in the Drosophila Visual System</a>
<strong>S. Lawrence Zipursky</strong></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-020-2879-3">Neuronal diversity and convergence in a visual system developmental atlas</a>
Claude Desplan NYU, 2020</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41467-025-56059-7">Morphological and functional convergence of visual projection neurons from diverse neurogenic origins in Drosophila</a>
Mehmet Neset Özel &amp; Claude Desplan, January 2025,Nature Communications</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-021-04262-z">Decoding gene regulation in the fly brain</a>
Stein Aerts, Nature, 2022</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-022-04564-w">A complete temporal <strong>transcription factor</strong> series in the fly visual system</a>
Claude Desplan</p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2025.06.04.657818v1.abstract">A High-Resolution Atlas of the Brain Predicts Lineage and Birth Order Underly Neuronal Identity</a>
Stephen F. Goodwin, 2025<br>
we generate the first high-resolution, single-cell transcriptomic atlas of the adult Drosophila melanogaster central brain by integrating multiple datasets, achieving an unprecedented tenfold coverage of every neuron in this complex tissue.</p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/50901">A genetic, genomic, and computational resource for exploring neural circuit function</a>
eLife, 2020
We used these tools to build a resource of high-resolution transcriptomes for 100 driver lines covering 67 cell types</p>
</li>
</ul>
<h4 id="other-gene-dataset">other gene dataset</h4>
<ul>
<li>
<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S001216062400160X">Isoforms of Terminal Selector Mamo Control Axon Guidance during Adult Drosophila Memory Center Construction via Semaphorin-1a</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s42003-023-04588-6">Cross-platform normalization enables machine learning model training on microarray and RNA-seq data simultaneously</a></p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/27041">Science Forum: The Human Cell Atlas</a></p>
</li>
</ul>
<h4 id="review">review</h4>
<ul>
<li><a href="https://www.nature.com/articles/s41593-024-01806-0">Opportunities and challenges of single-cell and spatially resolved genomics methods for neuroscience discovery</a>
two new axes of biological variation: cell-intrinsic regulation of cell states and expression programs and interactions between cells.<br>
nature neuroscience review 2024</li>
</ul>
<h4 id="analysis">analysis</h4>
<ul>
<li>
<p><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03152-z">SURGE: uncovering context-specific genetic-regulation of gene expression from single-cell RNA sequencing using latent-factor models</a></p>
</li>
<li>
<p><a href="https://academic.oup.com/nar/article/50/15/e86/6593121?login=false">Inference of cell state transitions and cell fate plasticity from single-cell with MARGARET</a></p>
</li>
<li>
<p><a href="https://onlinelibrary.wiley.com/doi/10.1002/ctm2.694">Single-cell RNA sequencing technologies and applications: A brief overview</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41576-023-00580-2">Methods and applications for single-cell and spatial multi-omics</a>
Nature Reviews Genetics (2023)</p>
</li>
<li>
<p><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5370-x">Gene2vec: distributed representation of genes based on co-expression</a></p>
</li>
</ul>
<h4 id="logical-gene-for-connectome">logical gene for connectome</h4>
<ul>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673531v1">Regulatory logic of neuronal identity specification in Drosophila</a><br>
Rose Coyne, McKenzie Treese, Yen-Chung Chen, ..., Claude Desplan, Mehmet Neset Özel, 2025, biorxiv</p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2006.11287">Discovering Symbolic Models from Deep Learning with Inductive Biases</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s42254-022-00518-3">On scientific understanding with artificial intelligence</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2204.10532">End-to-end symbolic regression with transformers</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2305.01582">interpretable Machine Learning for Science with PySR and SymbolicRegression.jl</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2404.19756">KAN: Kolmogorov-Arnold Networks</a></p>
</li>
</ul>
<h4 id="gene-regulatory-network">gene regulatory network</h4>
<ul>
<li><a href="https://www.sciencedirect.com/science/article/pii/S0070215320300363">Chapter Four - Gene regulatory networks during the development of the Drosophila visual system</a>
Yen-Chung Chen, Claude Desplan, 2020, Current Topics in Developmental Biology</li>
</ul>
<blockquote>
<p>recent advances in technology have allowed electrophysiological features and gene expression to be systematically compared and have demonstrated that similarity in gene expression correlates with similarity in firing patterns (Cadwell et al., 2016; Fuzik et al., 2016).
The visual system of Drosophila is an ideal platform to learn about gene regulatory networks and to bridge the gap between development and terminal features that establish diversity in the nervous system. First, the neurons of the visual system have been studied extensively for decades, and as a result, the morphology and the choice of neurotransmitter of a large proportion of the neurons are characterized (Fischbach, 1989; Raghu &amp; Borst, 2011; Raghu, Claussen, &amp; Borst, 2013; Varija Raghu, Reiff, &amp; Borst, 2011). Second, the high-resolution connectivity of most neuropils within the optic lobe has been actively determined by electron microscopy (Rivera-Alba et al., 2011; Shinomiya et al., 2019; Takemura et al., 2013, Takemura et al., 2015, Takemura et al., 2017).</p>
</blockquote>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s41586-021-04262-z">Decoding gene regulation in the fly brain</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s42256-022-00469-5">Inferring transcription factor regulatory networks from single-cell ATAC-seq data based on graph neural networks</a>
Nature Machine Intelligence, 2022</p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/19942">Genome-wide identification of neuronal activity-regulated genes in Drosophila</a> Brandeis
eLife, 2016</p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/abstract/document/8350274">Control of Gene Regulatory Networks Using Bayesian Inverse Reinforcement Learning</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s42003-022-03831-w">Shared regulation and functional relevance of local gene co-expression revealed by single cell analysis</a><br>
gene coexpression in scCOP and bulkCOP</p>
</li>
</ul>
<p class="katex-block"><span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><menclose notation="top bottom left right"><mtable rowspacing="0.16em" columnalign="center center center" columnlines="solid solid" columnspacing="1em" rowlines="solid solid"><mtr><mtd><mstyle scriptlevel="0" displaystyle="false"><mtext> Gene pairs </mtext></mstyle></mtd><mtd><mstyle scriptlevel="0" displaystyle="false"><mtext> scCOPs </mtext></mstyle></mtd><mtd><mstyle scriptlevel="0" displaystyle="false"><mrow><mi mathvariant="normal">¬</mi><mtext> scCOPs </mtext></mrow></mstyle></mtd></mtr><mtr><mtd><mstyle scriptlevel="0" displaystyle="false"><mtext> bulkCOPs </mtext></mstyle></mtd><mtd><mstyle scriptlevel="0" displaystyle="false"><mn>313</mn></mstyle></mtd><mtd><mstyle scriptlevel="0" displaystyle="false"><mn>2993</mn></mstyle></mtd></mtr><mtr><mtd><mstyle scriptlevel="0" displaystyle="false"><mrow><mi mathvariant="normal">¬</mi><mtext> bulkCOPs </mtext></mrow></mstyle></mtd><mtd><mstyle scriptlevel="0" displaystyle="false"><mn>3245</mn></mstyle></mtd><mtd><mstyle scriptlevel="0" displaystyle="false"><mn>232603</mn></mstyle></mtd></mtr></mtable></menclose></mrow><annotation encoding="application/x-tex">\begin{array}{|c|c|c|} \hline \text { Gene pairs } &amp; \text { scCOPs } &amp; \neg \text { scCOPs } \\
\hline \text { bulkCOPs } &amp; 313 &amp; 2993 \\ \hline \neg \text { bulkCOPs } &amp; 3245 &amp; 232603 \\ \hline \end{array}
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:3.64em;vertical-align:-1.55em;"></span><span class="mord"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:2.09em;"><span style="top:-4.05em;"><span class="pstrut" style="height:4.05em;"></span><span class="mtable"><span class="vertical-separator" style="height:3.6em;border-right-width:0.04em;border-right-style:solid;margin:0 -0.02em;vertical-align:-1.55em;"></span><span class="arraycolsep" style="width:0.5em;"></span><span class="col-align-c"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:2.05em;"><span style="top:-4.21em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord text"><span class="mord"> Gene pairs </span></span></span></span><span style="top:-3.01em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord text"><span class="mord"> bulkCOPs </span></span></span></span><span style="top:-1.81em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord">¬</span><span class="mord text"><span class="mord"> bulkCOPs </span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:1.55em;"><span></span></span></span></span></span><span class="arraycolsep" style="width:0.5em;"></span><span class="vertical-separator" style="height:3.6em;border-right-width:0.04em;border-right-style:solid;margin:0 -0.02em;vertical-align:-1.55em;"></span><span class="arraycolsep" style="width:0.5em;"></span><span class="col-align-c"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:2.05em;"><span style="top:-4.21em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord text"><span class="mord"> scCOPs </span></span></span></span><span style="top:-3.01em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord">313</span></span></span><span style="top:-1.81em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord">3245</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:1.55em;"><span></span></span></span></span></span><span class="arraycolsep" style="width:0.5em;"></span><span class="vertical-separator" style="height:3.6em;border-right-width:0.04em;border-right-style:solid;margin:0 -0.02em;vertical-align:-1.55em;"></span><span class="arraycolsep" style="width:0.5em;"></span><span class="col-align-c"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:2.05em;"><span style="top:-4.21em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord">¬</span><span class="mord text"><span class="mord"> scCOPs </span></span></span></span><span style="top:-3.01em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord">2993</span></span></span><span style="top:-1.81em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord">232603</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:1.55em;"><span></span></span></span></span></span><span class="arraycolsep" style="width:0.5em;"></span><span class="vertical-separator" style="height:3.6em;border-right-width:0.04em;border-right-style:solid;margin:0 -0.02em;vertical-align:-1.55em;"></span></span></span><span style="top:-2.5em;"><span class="pstrut" style="height:4.05em;"></span><span class="hline" style="border-bottom-width:0.04em;"></span></span><span style="top:-3.7em;"><span class="pstrut" style="height:4.05em;"></span><span class="hline" style="border-bottom-width:0.04em;"></span></span><span style="top:-4.9em;"><span class="pstrut" style="height:4.05em;"></span><span class="hline" style="border-bottom-width:0.04em;"></span></span><span style="top:-6.1em;"><span class="pstrut" style="height:4.05em;"></span><span class="hline" style="border-bottom-width:0.04em;"></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:1.55em;"><span></span></span></span></span></span></span></span></span></span></p>
<ul>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.12.19.629561v1">Multiomic foundation model predicts epigenetic regulation by zero-shot</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-021-03604-1">An atlas of gene regulatory elements in adult mouse cerebrum</a>
An anatomically comprehensive atlas of the adult human brain transcriptome
<a href="https://www.nature.com/articles/nature11405">https://www.nature.com/articles/nature11405</a></p>
</li>
</ul>
<h4 id="review-paper">review paper</h4>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s41576-023-00618-5">Gene regulatory network inference in the era of single-cell multi-omics</a>Julio Saez-Rodriguez
Nature Reviews Genetics, 2023</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41540-023-00312-6">Gene regulatory network reconstruction: harnessing the power of single-cell multi-omic data</a>
npj Systems Biology and Applications, review, 2023</p>
</li>
<li>
<p><a href="https://academic.oup.com/bib/article/22/3/bbaa190/5904505">A comprehensive survey of regulatory network inference methods using single cell RNA sequencing data</a>
Briefings in Bioinformatics, 2021</p>
</li>
</ul>
<h4 id="sota-models">sota models</h4>
<ul>
<li>
<p><a href="https://www.nature.com/articles/nmeth.4463">SCENIC: single-cell regulatory network inference and clustering</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41592-023-01938-4">SCENIC+: single-cell multiomic inference of enhancers and gene regulatory networks</a></p>
</li>
<li>
<p>LINGER: <a href="https://www.nature.com/articles/s41587-024-02182-7">Inferring gene regulatory networks from single-cell multiome data using atlas-scale external data</a></p>
</li>
</ul>
<p>-<a href="https://www.nature.com/articles/s41592-025-02918-6">Multimodal foundation transformer models for multiscale genomics</a><br>
Nature Methods (2025)</p>
<h4 id="application">application</h4>
<ul>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.03.16.435261v2">Encoding innate ability through a genomic bottleneck</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2407.15908">The Genomic Code: The genome instantiates a generative model of the organism</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2002.04461">TrajectoryNet: A Dynamic Optimal Transport Network for Modeling Cellular Dynamics</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S2211124719304012">A Comprehensive Drosophila melanogaster Transcription Factor Interactome</a></p>
</li>
</ul>
<h3 id="gene-normalization">gene normalization</h3>
<ul>
<li>
<p><a href="https://scanpy.readthedocs.io/en/stable/">Scanpy – Single-Cell Analysis in Python</a></p>
</li>
<li>
<p><a href="https://scanpy.readthedocs.io/en/stable/tutorials/basics/clustering.html">scanpy Preprocessing and clustering</a></p>
</li>
<li>
<p><a href="https://scanpy.readthedocs.io/en/stable/generated/scanpy.pp.highly_variable_genes.html">scanpy highly_variable_genes</a></p>
</li>
<li>
<p><a href="https://satijalab.org/seurat/reference/normalizedata">seurat Normalize Data</a></p>
</li>
<li>
<p><a href="https://bioconductor.org/packages/release/workflows/vignettes/RNAseq123/inst/doc/limmaWorkflow.html">RNA-seq analysis is easy as 1-2-3 with limma, Glimma and edgeR</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/nbt.4096">Integrating single-cell transcriptomic data across different conditions, technologies, and species</a></p>
</li>
</ul>
<h3 id="functional-and-structure-conenctome">functional and structure conenctome</h3>
<ul>
<li>
<p><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3002314">Integrating multimodal and multiscale connectivity blueprints of the human cerebral cortex in health and disease</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41593-024-01812-2">The cell-type underpinnings of the human functional cortical connectome</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/nrn2575">Complex brain networks: graph theoretical analysis of structural and functional systems</a>
Ed Bullmore &amp; Olaf Sporns (2009). Nature Reviews Neuroscience</p>
</li>
<li>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(13)00847-4">Intrinsic coupling modes: multiscale interactions in ongoing brain activity</a>
Engel, A. K., Gerloff, C., Hilgetag, C. C., &amp; Nolte, G. (2013). Neuron, 80(4), 867-886.</p>
</li>
<li>
<p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/mrm.1910340409">Functional connectivity in the motor cortex of resting human brain using echo-planar mri</a>
Biswal, B., F. Zerrin Yetkin, Victor M. Haughton, James S. Hyde (1995). Magnetic Resonance in Medicine 34(4): 537-541.</p>
</li>
<li>
<p><a href="https://academic.oup.com/cercor/article/24/1/17/311346">A weighted and directed interareal connectivity matrix for macaque cerebral cortex</a>
Markov NT, et al. (2014). Cereb Cortex 24:17–36.</p>
</li>
<li>
<p><a href="https://www.jneurosci.org/content/36/21/6758">Using diffusion tractography to predict cortical connection strength and distance: a quantitative comparison with tracers in the monkey</a>
Donahue, C.J., Sotiropoulos, S.N., Jbabdi, S., Hernandez-Fernandez, M., Behrens, T.E., Dyrby, T.B., Coalson, T., Kennedy, H., Knoblauch, K., Van Essen, D.C., Glasser, M.F. (2016). J. Neurosci. 36, 6758–6770.</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1053811910002344">Can structure predict function in the human brain?</a>
Honey, C., Thivierge JP, Sporns O (2010). Neuroimage 52(3): 766-776.</p>
</li>
<li>
<p><a href="https://www.jneurosci.org/content/34/23/7910">Identification of optimal structural connectivity using functional connectivity and neural modeling</a>
Deco, G., McIntosh, A. R., Shen, K., Hutchison, R. M., Menon, R. S., Everling, S., Hagmann, P., &amp; Jirsa, V. K. (2014). The Journal of neuroscience, 34(23), 7910–7916.</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1053811913000260">Evaluation and calibration of functional network modeling methods based on known anatomical connections</a>
Dawson, DA, Cha K, Lewis LB, Mendola JD, Shmuel A (2013). Neuroimage 67: 331-343.</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S1361841522001104">Predicting brain structural network using functional connectivity</a>
multi-GCN based GAN, AD (2022). Medical Image Analysis, 79, 102463.</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1053811923006456">Prediction of brain sex from EEG: using large-scale heterogeneous dataset for developing a highly accurate and interpretable ML model</a>
(2023). NeuroImage</p>
</li>
<li>
<p><a href="https://www.jneurosci.org/content/45/7/e1510242024#page=14.33">Topographic Axes of Wiring Space Converge to Genetic Topography in Shaping the Human Cortical Layout</a>
(2024). Journal of Neuroscience</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S1053811919300114">A practical guide to linking brain-wide gene expression and neuroimaging data</a>
(2019). NeuroImage</p>
</li>
</ul>
<h3 id="other-connectome-model">other connectome model:</h3>
<ul>
<li><a href="https://openreview.net/forum?id=dqWobzlAGb">Modelling brain connectomes networks: Solv is a worthy competitor to hyperbolic geometry!</a></li>
</ul>
<h2 id="reinforcement-learning-rl">Reinforcement learning (RL)</h2>
<ul>
<li>
<p><a href="https://arxiv.org/abs/2103.08228">Learning Symbolic Rules for Interpretable Deep Reinforcement Learning</a></p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/abstract/document/10650177">Applying Logical Rules to Reinforcement Learning for Interpretable Knowledge Graph Reasoning</a></p>
</li>
<li>
<p><a href="https://link.springer.com/chapter/10.1007/978-3-540-87993-0_27">A Logical Framework to Reinforcement Learning Using Hybrid Probabilistic Logic Programs</a></p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/document/8431181">A Policy Search Method For Temporal Logic Specified Reinforcement Learning Tasks</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2106.11417">Interpretable Model-based Hierarchical Reinforcement Learning using Inductive Logic Programming</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2308.06094">Reinforcement Logic Rule Learning for Temporal Point Processes</a></p>
</li>
</ul>
<h2 id="graph">graph</h2>
<ul>
<li>
<p>JC: <a href="https://docs.google.com/spreadsheets/d/10R6poOIIqA4OCsiSGs5mK4Zkq9pm8aPxk1IQ3Zl0NPU">https://docs.google.com/spreadsheets/d/10R6poOIIqA4OCsiSGs5mK4Zkq9pm8aPxk1IQ3Zl0NPU</a></p>
</li>
<li>
<p>datasests: <a href="https://github.com/gephi/gephi/wiki/datasets">Gephi sample datasets</a></p>
</li>
<li>
<p>DGL 2.0: <a href="https://www.dgl.ai/">Streamlining Your GNN Data Pipeline from Bottleneck to Boost</a><br>
<a href="https://github.com/dmlc/dgl/blob/master/examples/README.md#agnn">github</a></p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/document/4700287">The Graph Neural Network Model</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1611.07308">Variational Graph Auto-Encoders</a>
Bayesian Deep Learning Workshop (NIPS 2016)  <a href="https://github.com/DaehanKim/vgae_pytorch">code</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1710.10903">Graph Attention Networks</a>
Yoshua Bengio</p>
</li>
<li>
<p><a href="https://petar-v.com/GAT/">Graph Attention Networks</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1803.03735">Attention-based Graph Neural Network for Semi-supervised Learning</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2407.11907">GraphFM: A Scalable Framework for Multi-Graph Pretraining</a></p>
</li>
<li>
<p><a href="https://github.com/gongchenghua/Papers-Graphs-with-Heterophily">A Survey on Learning from Graphs with Heterophily: Recent Advances and Future Directions</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2412.00742">Revisiting Self-Supervised Heterogeneous Graph Learning from Spectral Clustering Perspective</a>
within-community connection prediction</p>
</li>
</ul>
<h3 id="link-prediction">Link prediction</h3>
<p><a href="https://paperswithcode.com/task/link-prediction">https://paperswithcode.com/task/link-prediction</a></p>
<ul>
<li>
<p><a href="https://link.springer.com/chapter/10.1007/978-3-642-23783-6_28">Link Prediction via Matrix Factorization</a></p>
</li>
<li></li>
<li>
<p>GraphSAGE: <a href="https://arxiv.org/abs/1706.02216v4">Inductive Representation Learning on Large Graphs</a>
GraphSAGE, a general, inductive framework that leverages node feature information (e.g., text attributes) to efficiently generate node embeddings for previously unseen data.</p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/document/9787711">Inductive Link Prediction via Interactive Learning Across Relations in Multiplex Networks</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S1751157721000493">Community-guided link prediction in <strong>multiplex</strong> networks</a></p>
</li>
</ul>
<h4 id="review-1">review</h4>
<ul>
<li>
<p><a href="https://dl.acm.org/doi/10.1145/3012704">A Survey of Link Prediction in Complex Networks</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S037843711000991X">Link prediction in complex networks: A survey</a></p>
</li>
</ul>
<h3 id="graph-embedding">Graph Embedding</h3>
<ul>
<li>
<p><a href="https://arxiv.org/abs/1503.03578">LINE: Large-scale Information Network Embedding</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1903.00757">GraphVite - graph embedding at high speed and large scale</a><br>
<a href="https://github.com/DeepGraphLearning/graphvite">GraphVite github</a></p>
</li>
</ul>
<h3 id="review-2">review</h3>
<ul>
<li>
<p><a href="https://arxiv.org/abs/1709.07604">A Comprehensive Survey of Graph Embedding</a></p>
</li>
<li>
<p><a href="https://dl.acm.org/doi/10.1145/3154524">Network Structure Inference, A Survey: Motivations, Methods, and Applications</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1901.00596">A Comprehensive Survey on Graph Neural Networks</a>
Cited by 11155</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S2666651021000012">Graph neural networks: A review of methods and applications</a>
Cited by 6966</p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/abstract/document/9416834">Graph Learning: A Survey</a></p>
</li>
<li>
<p><a href="https://paperswithcode.com/methods/category/graph-models">papers with code: Graph Models</a></p>
</li>
</ul>
<h3 id=""></h3>
<ul>
<li><a href="https://arxiv.org/abs/2112.14936">Are we really making much progress? Revisiting, benchmarking, and refining heterogeneous graph neural networks</a>
we construct the Heterogeneous Graph Benchmark (HGB), consisting of 11 diverse datasets with three tasks.</li>
</ul>
<pre><code>| Dataset | Nodes | Node Types | Edges | Edge Types | Target |
|---------|-------|------------|-------|------------|--------|
| Amazon  | 10,099| 1          | 148,659| 2          | product-product |
| LastFM  | 20,612| 3          | 141,521| 3          | user-artist |
| PubMed  | 63,109| 4          | 244,986| 10         | disease-disease |
</code></pre>
<h3 id="disease-gene">disease gene</h3>
<ul>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0925231221011012">GANLDA: Graph attention network for lncRNA-disease associations prediction</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2022.11.21.517436v1">Genome-wide prediction of dominant and recessive neurodevelopmental disorder risk genes</a></p>
</li>
<li>
<p><a href="https://www.cell.com/ajhg/fulltext/S0002-9297(20)30108-7">Mantis-ml: Disease-Agnostic Gene Prioritization from High-Throughput Genomic Screens by Stochastic Semi-supervised Learning</a></p>
</li>
</ul>
<h3 id="graph-and-connectome">graph and connectome</h3>
<p>BrainGNN: Interpretable Brain Graph Neural Network for fMRI Analysis
<a href="https://www.sciencedirect.com/science/article/abs/pii/S1361841521002784">https://www.sciencedirect.com/science/article/abs/pii/S1361841521002784</a></p>
<p>Graph Neural Networks for Brain Graph Learning: A Survey
<a href="https://arxiv.org/abs/2406.02594">https://arxiv.org/abs/2406.02594</a>
2024</p>
<h3 id="other">other</h3>
<ul>
<li>
<p><a href="https://royalsocietypublishing.org/doi/abs/10.1098/rspb.1967.0013#">The frequency of nerve action potentials generated by applied currents</a>
R. B. Stein and Alan Lloyd Hodgkin</p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/abstract/document/9933612">Fractional Spiking Neuron: Fractional Leaky Integrate-and-Fire Circuit Described with Dendritic Fractal Model</a></p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/abstract/document/4120609">A Dendritic Compartment Model Neuron</a></p>
</li>
</ul>
<h3 id="olfactory-system">olfactory system</h3>
<ul>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0893608016301666">A computational model of conditioning inspired by Drosophila olfactory system</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41598-021-85841-y">Predictive olfactory learning in Drosophila</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2025.02.15.638426v1.abstract">A vector-based strategy for <strong>olfactory</strong> navigation in Drosophila</a>  L.F. Abbott, Vanessa Ruta</p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.10.05.616803v1.abstract">Neural dynamics for working memory and evidence integration during olfactory navigation in Drosophila</a></p>
</li>
<li>
<p><a href="https://www.cell.com/cell/fulltext/S0092-8674(23)00472-5">Generating parallel representations of position and identity in the olfactory system</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0960982209013013">Olfactory Information Processing in Drosophila</a> Gregory S.X.E. Jefferis, 2009</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/nn.2489">Diversity and wiring variability of olfactory local interneurons in the Drosophila antennal lobe</a></p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/66018">Information flow, cell types and stereotypy in a full <strong>olfactory</strong> connectome</a>
Gregory S X E Jefferis</p>
</li>
<li>
<p><a href="https://www.frontiersin.org/journals/neural-circuits/articles/10.3389/fncir.2020.561822/full">Subpopulations of Projection Neurons in the Olfactory Bulb</a>
REVIEW article,Front. Neural Circuits, 27 August 2020</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/nature10926">Teneurins instruct synaptic partner matching in an olfactory map</a>
Weizhe Hong, Timothy J. Mosca &amp; Liqun Luo<br>
Teneurins instruct matching specificity between synaptic partners through homophilic attraction.</p>
</li>
</ul>
<h3 id="mean-field">mean field</h3>
<ul>
<li>
<p><a href="https://link.springer.com/article/10.1023/A:1008925309027">Dynamics of Sparsely Connected Networks of Excitatory and Inhibitory Spiking Neurons</a>
Brunel, 2000, JofCNS</p>
</li>
<li>
<p><a href="https://journals.aps.org/prx/abstract/10.1103/PhysRevX.5.021028">Macroscopic Description for Networks of Spiking Neurons</a>
Ernest Montbrió, Diego Pazó, and Alex Roxin, 2015<br>
Quadratic IF neuron</p>
</li>
<li>
<p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005507">Towards a theory of cortical columns: From spiking neurons to interacting neural populations of finite size</a>
Schwalger etal., 2017</p>
</li>
<li>
<p><a href="https://academic.oup.com/cercor/article/24/3/785/398560">The Cell-Type Specific Cortical Microcircuit: Relating Structure and Activity in a Full-Scale Spiking Network Model</a>
Potjans &amp; Diesmann, 2014, Cerebral Cortex<br>
Mean field for Mesocsopic</p>
</li>
</ul>
<h2 id="neuroai">NeuroAI</h2>
<ul>
<li>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(22)00806-6#%20">In vitro neurons learn and exhibit sentience when embodied in a simulated game-world</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41928-023-01069-w">Brain organoid reservoir computing for artificial intelligence</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2212.13345">The Forward-Forward Algorithm: Some Preliminary Investigations</a>
Geoffrey Hinton</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0959438818302174">Constraining computational models using electron microscopy wiring diagrams</a>
Ashok Litwin-Kumar , Srinivas C Turaga<br>
Review, Current Opinion in Neurobiology, 2019</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41592-024-02497-y">NeuroMechFly v2: simulating embodied <strong>sensorimotor</strong> control in adult Drosophila</a>  Pavan Ramdya</p>
</li>
</ul>
<h3 id="connectome--ann">Connectome &amp; ANN</h3>
<p><a href="connectome%20inspired%20neural%20network.html">connectome inspired neural network</a><br>
html <a href="https://lengyuner.github.io/blog/connectome%20inspired%20neural%20network.html">connectome inspired neural network</a></p>
<ul>
<li>
<p><a href="https://arxiv.org/abs/1912.08986">Deep Connectomics Networks: Neural Network Architectures Inspired by Neuronal Networks</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2503.00073">Forecasting Whole-Brain Neuronal Activity from Volumetric Video</a>
ZAPBench</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-025-08829-y">Foundation model of neural activity predicts response to new stimulus types</a>
Eric Y. Wang, ... , Xaq Pitkow, Fabian H. Sinz &amp; Andreas S. Tolias, 2025, Nature</p>
</li>
</ul>
<h3 id="geometry">geometry</h3>
<ul>
<li><a href="https://openreview.net/forum?id=dqWobzlAGb">Modelling brain connectomes networks: Solv is a worthy competitor to hyperbolic geometry!</a>
ICLR 2024 Conference Submission</li>
</ul>
<h3 id="brain-simulation--activity-prediction">Brain simulation / activity prediction</h3>
<ul>
<li><a href="https://openreview.net/forum?id=7ntI4kcoqG">AMAG: Additive, Multiplicative and Adaptive Graph Neural Network For Forecasting Neuron Activity</a>
NeurIPS 2023 poster</li>
</ul>
<h4 id="model">model</h4>
<ul>
<li><a href="https://arxiv.org/abs/2002.09405">Learning to Simulate Complex Physics with Graph Networks</a><br>
为了做这个，我们需要什么样的data。有wide field and the 2p microscope，是不是就可以模拟一些基础规律了</li>
</ul>
<h3 id="network-analysis">Network analysis</h3>
<p>BOOK <a href="https://www.sciencedirect.com/book/9780124079083/fundamentals-of-brain-network-analysis">Fundamentals of Brain Network Analysis</a></p>
<p>BOOK <a href="https://www.sciencedirect.com/science/article/pii/B9780124079083000030">Chapter 3 - Connectivity Matrices and Brain Graphs</a></p>
<p>Review <a href="https://www.frontiersin.org/articles/10.3389/fnsys.2020.609316/full">Neural Networks With Motivation</a></p>
<ul>
<li><a href="https://www.researchgate.net/publication/2267079_Modeling_Small_Networks">Modeling small networks</a>
1998, L F Abbott, Eve Marder</li>
</ul>
<h3 id="synapse-plasticity">synapse plasticity</h3>
<h4 id="stdp">STDP</h4>
<p>nearest neighbor STDP (Izhikevich, 2003)</p>
<ul>
<li>
<p><a href="https://doi.org/10.1162/089976603321891783">Relating STDP to BCM</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2310.20309">Tensor formalism for predicting synaptic connections with ensemble modeling or optimization</a><br>
Tirthabir Biswas, Tianzhi Lambus Li, James E. Fitzgerald</p>
</li>
<li>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(15)00210-X">Inhibitory and Excitatory Spike-Timing-Dependent Plasticity in the Auditory Cortex</a>
Inhibitory potentiation occurs when either pre- or postsynaptic spikes come first<br>
Excitatory and inhibitory inputs become bound together by postsynaptic spiking</p>
</li>
</ul>
<h4 id="btsp">BTSP</h4>
<ul>
<li><a href="https://www.science.org/doi/10.1126/science.aan3846">Behavioral time scale synaptic plasticity underlies CA1 place fields</a>
A different form of synaptic plasticity</li>
</ul>
<h2 id="neural-data-recording">neural data recording</h2>
<h3 id="whole-brain-imaging">whole brain imaging</h3>
<h4 id="drosophila-data">drosophila data</h4>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s41467-023-41261-2">The spatial and temporal structure of neural activity across the fly brain</a>
swept confocally-aligned planar excitation (SCAPE) microscopy,<br>
1419 ± 78 stable, single-cell ROIs per animal, contained nearly all neurons within 70 μm of the dorsal surface<br>
<a href="https://springernature.figshare.com/articles/dataset/Flygenvectors_The_spatial_and_temporal_structure_of_neural_activity_across_the_fly_brain/23749074">https://springernature.figshare.com/articles/dataset/Flygenvectors_The_spatial_and_temporal_structure_of_neural_activity_across_the_fly_brain/23749074</a></p>
</li>
<li>
<p><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(23)01763-3">Mapping the neural dynamics of locomotion across the Drosophila brain</a></p>
</li>
</ul>
<p>Thomas R. Clandinin, 2024, Current Bilogy</p>
<ul>
<li>
<p><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(21)00343-2">The connectome predicts resting-state functional connectivity across the Drosophila brain</a>
Thomas R. Clandinin, 2021, Current Bilogy</p>
</li>
<li>
<p><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(17)30813-8">Whole-Brain Calcium Imaging Reveals an Intrinsic Functional Network in Drosophila</a>
Current Bilogy
<a href="https://data.mendeley.com/datasets/8b6nw2xxhn/1">https://data.mendeley.com/datasets/8b6nw2xxhn/1</a></p>
</li>
</ul>
<p>Thomas R. Clandinin, 2017, Current Bilogy</p>
<h4 id="other-data">other data</h4>
<ul>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2025.01.27.635185v1.full">In vivo autofluorescence lifetime imaging of the Drosophila brain captures metabolic shifts associated with memory formation</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41467-023-41261-2">The spatial and temporal structure of neural activity across the fly brain</a>
Evan S. Schaffer ... L. F. Abbott &amp; Richard Axel, 2023, Nature Communications</p>
</li>
</ul>
<p>We used swept, confocally-aligned planar excitation (SCAPE) microscopy to image all cells in a large volume of the brain of adult Drosophila with high spatiotemporal resolution while flies engaged in a variety of spontaneous behaviors.</p>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s42254-022-00430-w?fromPaywallRec=false">Imaging whole-brain activity to understand behaviour</a>
Review Article</p>
</li>
<li>
<p><a href="https://www.science.org/doi/full/10.1126/science.adf9725">Wide-field fluorescence lifetime imaging of neuron spiking and subthreshold activity in vivo</a></p>
</li>
<li>
<p><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(23)01763-3">Mapping the neural dynamics of locomotion across the Drosophila brain</a></p>
</li>
<li>
<p><a href="https://www.pnas.org/doi/full/10.1073/pnas.0905267106">Correspondence of the brain's functional architecture during activation and rest</a></p>
</li>
<li>
<p><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2006732">Fast near-whole–brain imaging in adult Drosophila during responses to stimuli and behavior</a>
Plos Biology</p>
</li>
<li>
<p><a href="https://www.cell.com/iscience/fulltext/S2589-0042(19)30091-4">Imaging through the Whole Brain of Drosophila at λ/20 Super-resolution</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-021-03497-0">Coupling of activity, metabolism and behaviour across the Drosophila brain</a> Surya Ganguli &amp; Thomas R. Clandinin</p>
</li>
<li>
<p><a href="https://www.cell.com/cell/article/S0092-8674(24)00830-4/fulltext">Multiphoton fluorescence microscopy for in vivo imaging</a>
Na Ji <a href="mailto:jina@berkeley.edu">jina@berkeley.edu</a>, 2024</p>
</li>
</ul>
<h3 id="cell-tracking">cell tracking</h3>
<ul>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2020.06.22.165696v3.full">EMC2: A versatile algorithm for robust tracking of calcium dynamics from individual neurons in behaving animals</a></p>
</li>
<li>
<p><a href="https://elifesciences.org/articles/59187">3DeeCellTracker, a deep learning-based pipeline for segmenting and tracking cells in 3D time lapse images</a></p>
</li>
</ul>
<h3 id="wormid">wormid</h3>
<ul>
<li>
<p><a href="https://arxiv.org/abs/2503.07348">Cycle-Consistent Multi-Graph Matching for Self-Supervised Annotation of C.Elegans</a>
2025</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S2667237524003540">Unifying community whole-brain imaging datasets enables robust neuron identification and reveals determinants of neuron position in C. elegans</a>
Daniel Y. Sprague,..., Saul Kato</p>
</li>
</ul>
<h2 id="neural-data-analysis--modeling">neural data analysis &amp; modeling</h2>
<ul>
<li>
<p><a href="https://direct.mit.edu/neco/article/27/1/104/8035/Spatiotemporal-Conditional-Inference-and">Spatiotemporal conditional inference and hypothesis tests for neural ensemble spiking precision</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0959438816302689">Maximum entropy models as a tool for building precise neural controls</a>
Cristina Savin, Gašper Tkačik</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-025-09180-y">Unsupervised pretraining in biological neural networks</a>
Lin Zhong, Scott Baptista, Rachel Gattoni, Jon Arnold, Daniel Flickinger, Carsen Stringer &amp; Marius Pachitariu
Nature (2025)</p>
</li>
</ul>
<h3 id="reivew">reivew</h3>
<ul>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0959438817302428">Neural data science: accelerating the experiment-analysis-theory cycle in large-scale neuroscience </a>
ICCV, 2019</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41583-025-00991-6">Recording single neurons at scale in the primate brain</a>
Frank F. Lanfranchi, 2025, Nature Reviews Neuroscience</p>
</li>
<li>
<p><a href="https://www.science.org/doi/full/10.1126/science.adp7429">Analysis methods for large-scale neuronal recordings</a>
Carsen Stringer, Marius Pachitariu</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41583-025-00992-5">How distributed is the brain-wide network that is recruited for cognition?</a></p>
</li>
<li></li>
</ul>
<h3 id="high-dimensional-data">high dimensional data</h3>
<ul>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.aav7893">Spontaneous behaviors drive multidimensional, brainwide activity</a>
Carsen Stringer</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-019-1346-5">High-dimensional geometry of population responses in visual cortex</a>
Carsen Stringer</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0896627324001211">Simultaneous, cortex-wide dynamics of up to <strong>1 million neurons</strong> reveal unbounded scaling of dimensionality with neuron number</a></p>
</li>
</ul>
<h3 id="dimensional-reduction">dimensional reduction</h3>
<ul>
<li>
<p>Phate: <a href="https://www.nature.com/articles/s41587-019-0336-3">Visualizing structure and transitions in high-dimensional biological data</a>
Phate</p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1608.06315">LFADS - Latent Factor Analysis via Dynamical Systems</a></p>
</li>
</ul>
<h3 id="manifold-reconstriction">manifold reconstriction</h3>
<ul>
<li>
<p><a href="https://arxiv.org/abs/2012.13804">Approximation of Functions on Manifolds in High Dimension from Noisy Scattered Data</a>
<strong>Manifold Locally Optimal Projection (MLOP)</strong></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2102.01750">Manifold Repairing, Reconstruction and Denoising from Scattered Data in High-Dimension</a>
<strong>Repairing Manifold Locally Optimal Projection (R-MLOP)</strong></p>
</li>
<li>
<p><a href="https://www.cs.tau.ac.il/~dcor/articles/2007/Parameterization-free-Projection.pdf">Parameterization-free Projection for Geometry Reconstruction</a></p>
</li>
</ul>
<h3 id="activity-analysis">activity analysis</h3>
<ul>
<li>
<p>Meta Neural Band: <a href="https://www.nature.com/articles/s41586-025-09255-w">A generic non-invasive neuromotor interface for human-computer interaction</a> Meta, 2025, Nature
surface electromyography (sEMG)</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-023-06683-4">Neural signal propagation atlas of Caenorhabditis elegans</a> Andrew M. Leifer</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41593-024-01845-7">Dynamical constraints on neural population activity</a>
Byron M. Yu &amp; Aaron P. Batista</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-023-06714-0">Preserved neural dynamics across animals performing similar behaviour</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41593-024-01845-7">Dynamical constraints on neural population activity</a>
Byron M. Yu &amp; Aaron P. Batista</p>
</li>
</ul>
<h3 id="head-direction">head direction</h3>
<ul>
<li>
<p><a href="https://link.springer.com/article/10.1007/s00359-023-01615-z">Global inhibition in head-direction neural circuits: a systematic comparison between connectome-based spiking neural circuit models</a>
Ning Chang, Hsuan-Pei Huang &amp; Chung-Chuan Lo (2023)</p>
</li>
<li>
<p><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011913">Estimating orientation in natural scenes: A spiking neural network model of the insect central complex</a>
Rachael Stentiford ,James C. Knight,Thomas Nowotny,Andrew Philippides,Paul Graham 2024</p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2025.05.26.655406v1.abstract">From Synapses to Dynamics: Obtaining Function from Structure in a Connectome Constrained Model of the Head Direction Circuit</a>
Sunny Duan, Ling Liang Dong, Ila Fiete, 2025</p>
</li>
</ul>
<h2 id="encoding-and-decoding">encoding and decoding</h2>
<ul>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.1234330">Neural Decoding of Visual Imagery During Sleep</a></p>
</li>
<li>
<p><a href="https://www.jneurosci.org/content/37/14/3764.abstract">Locomotion Enhances Neural Encoding of Visual Stimuli in Mouse V1</a></p>
</li>
<li>
<p><a href="https://journals.physiology.org/doi/full/10.1152/jn.01046.2012">Bayesian decoding using unsorted spikes in the rat hippocampus</a>
Fabian Kloosterman, Stuart P. Layton, Zhe Chen, and Matthew A. Wilson</p>
</li>
<li>
<p><a href="https://www.cell.com/cell/fulltext/S0092-8674(24)00980-2">Decoding the brain: From neural representations to mechanistic models</a></p>
</li>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.adh7814">Hebbian instruction of axonal connectivity by endogenous correlated spontaneous activity</a></p>
</li>
</ul>
<h3 id="deep-learning-method">deep learning method</h3>
<ul>
<li>
<p>NEMO: <a href="https://www.biorxiv.org/content/10.1101/2024.11.05.622159v1">In vivo cell-type and brain region classification via multimodal contrastive learning</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2402.17772">EEG2Rep: Enhancing Self-supervised EEG Representation Through Informative Masked Inputs</a></p>
</li>
<li>
<p><a href="https://openreview.net/forum?id=90O5cvFZkZ">GUST: Combinatorial Generalization by Unsupervised Grouping with Neuronal Coherence</a>
NeurIPS 2023 poster</p>
</li>
</ul>
<h4 id="fundation-model">fundation model</h4>
<ul>
<li>
<p>LFADS:<a href="https://www.nature.com/articles/s41592-018-0109-9">Inferring single-trial neural population dynamics using sequential auto-encoders</a> LFADS <a href="https://arxiv.org/abs/1608.06315">(method part)</a></p>
</li>
<li>
<p>MtM: <a href="https://openreview.net/forum?id=nRRJsDahEg">Towards a &quot;Universal Translator&quot; for Neural Dynamics at Single-Cell, Single-Spike Resolution</a>
Yizi, Liam, Cole</p>
</li>
<li>
<p><a href="https://poyo-brain.github.io/">POYO-1: A Unified, Scalable Framework for Neural Population Decoding</a></p>
</li>
</ul>
<h4 id="diffusion-model-visual-stimulus">diffusion model/ visual Stimulus</h4>
<ul>
<li>
<p><a href="https://link.springer.com/chapter/10.1007/978-3-642-00582-4_2">Decoding What People See from Where They Look: Predicting Visual Stimuli from Scanpaths</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41598-023-42891-8">Natural scene reconstruction from fMRI signals using generative latent diffusion</a>
Scientific Report, 2023</p>
</li>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.1234330">Neural Decoding of Visual Imagery During Sleep</a>
Science, 2013</p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2211.06956">Seeing Beyond the Brain: Conditional Diffusion Model with Sparse Masked Modeling for Vision Decoding</a>
CVPR, 2023</p>
</li>
</ul>
<h2 id="behavior">behavior</h2>
<ul>
<li>
<p><a href="https://arxiv.org/abs/2311.04069"><strong>LISBET</strong>: a machine learning model for the automatic segmentation of social behavior motifs</a></p>
</li>
<li>
<p>GLM-HMM: <a href="https://www.nature.com/articles/s41593-019-0533-x">Unsupervised identification of the internal states that shape natural behavior</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41593-021-01007-z">Mice alternate between discrete strategies during perceptual decision-making</a>
Jonathan W. Pillow</p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2007.08723">End-to-end Deep Prototype and Exemplar Models for Predicting Human Behavior</a>
Thomas L. Griffiths</p>
</li>
<li>
<p><a href="https://www.frontiersin.org/articles/10.3389/fncir.2012.00108/full">Spatial vision in insects is facilitated by shaping the dynamics of visual input through behavioral action</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/nature14855">Arithmetic and local circuitry underlying dopamine prediction errors</a></p>
</li>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.275.5306.1593">A Neural Substrate of Prediction and Reward</a></p>
</li>
<li>
<p><a href="https://openreview.net/forum?id=nosngu5XwY9">Dynamic Inverse Reinforcement Learning for Characterizing Animal Behavior</a>  Zoe Ashwood, Aditi Jha, Jonathan W. Pillow<br>
NeurIPS 2022</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-022-05611-2">Spontaneous behaviour is structured by reinforcement without explicit reward</a>
Sandeep Robert Datta</p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2025.01.08.631866v1.abstract">The spinal premotor network driving scratching flexor and extensor alternation</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41592-024-02319-1">Lightning Pose: improved animal pose estimation via semi-supervised learning, Bayesian ensembling and cloud-native open-source tools</a>
Liam Paninski, 2024, Nature Methods</p>
</li>
</ul>
<h3 id="oscilation">oscilation</h3>
<ul>
<li><a href="https://www.sciencedirect.com/science/article/pii/S0896627323002143">Brain rhythms have come of age</a></li>
</ul>
<h3 id="place-cell-and-grid-cell">place cell and grid cell</h3>
<ul>
<li>
<p><a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/medial-entorhinal-cortex">medial entorhinal cortex</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S009286742031388X">The Tolman-Eichenbaum Machine: Unifying Space and Relational Memory through Generalization in the Hippocampal Formation</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.10.28.466284v5">Emergence of robust global modules from local interactions and smooth gradients</a></p>
</li>
</ul>
<h2 id="vision">vision</h2>
<ul>
<li>
<p><a href="https://www.pnas.org/doi/full/10.1073/pnas.1922294117">The limits of color awareness during active, real-world vision</a>
using VR to test the color fade perception</p>
</li>
<li>
<p><a href="https://www.frontiersin.org/articles/10.3389/fncir.2015.00033/full">A common evolutionary origin for the ON- and OFF-edge motion detection pathways of the Drosophila visual system</a>
Ian A. Meinertzhagen</p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.12.20.473513v1">Neuronal circuits integrating visual motion information in Drosophila</a>
T4 and T5 neuron</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-08255-6">Social state alters vision using three circuit mechanisms in Drosophila</a>
Vanessa Ruta &amp; Gerald M. Rubin</p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.05.12.540591v2">Deep learning-driven characterization of single cell tuning in primate visual area V4 supports topological organization</a></p>
</li>
</ul>
<h3 id="drosophila">drosophila</h3>
<ul>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627308007058">The Neural Substrate of Spectral Preference in Drosophila</a></p>
</li>
<li>
<p><a href="https://www.cell.com/current-biology/fulltext/S0960-9822(22)01023-5">Neuronal circuits integrating visual motion information in Drosophila melanogaster</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41467-025-56059-7">Morphological and functional convergence of visual projection neurons from diverse neurogenic origins in Drosophila</a></p>
</li>
</ul>
<h2 id="others">others</h2>
<h3 id="diffusion-model">diffusion model</h3>
<p>Diffusion and Score-Based Generative Models by Yang Song, Stanford University
<a href="https://www.youtube.com/watch?v=wMmqCMwuM2Q&amp;ab_channel=MITCBMM">https://www.youtube.com/watch?v=wMmqCMwuM2Q&amp;ab_channel=MITCBMM</a></p>
<ul>
<li><a href="https://arxiv.org/abs/1503.03585">Deep Unsupervised Learning using Nonequilibrium Thermodynamics</a> Surya Ganguli</li>
</ul>
<h3 id="ml">ML</h3>
<ul>
<li>
<p><a href="https://www.pure.ed.ac.uk/ws/portalfiles/portal/322711194/Logic_Reinforcement_BUEFF_DOA22122022_AFV_CC_BY.pdf">Logic + Reinforcement Learning + Deep Learning: A Survey</a>
Review, 2023</p>
</li>
<li>
<p><a href="https://link.springer.com/article/10.1007/s10994-024-06538-7">Learning explanatory logical rules in non-linear domains: a neuro-symbolic approach</a>
2024</p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/document/10650177">Applying Logical Rules to Reinforcement Learning for Interpretable Knowledge Graph Reasoning</a></p>
</li>
<li>
<p><a href="https://proceedings.mlr.press/v97/jiang19a.html">Neural Logic Reinforcement Learning</a></p>
</li>
</ul>
<h3 id="hippocamus">hippocamus</h3>
<ul>
<li>
<p><a href="https://www.jneurosci.org/content/28/52/14271.short">Network Dynamics Underlying the Formation of Sparse, Informative Representations in the Hippocampus</a></p>
</li>
<li>
<p><a href="https://www.pnas.org/doi/abs/10.1073/pnas.1408365111">Sparse and distributed coding of episodic memory in neurons of the human hippocampus</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0166432819303936">Synaptic properties of newly generated granule cells support sparse coding in the adult hippocampus</a></p>
</li>
</ul>
<h3 id="gene">gene</h3>
<ul>
<li>
<p><a href="https://arxiv.org/abs/2306.15794">HyenaDNA: Long-Range Genomic Sequence Modeling at Single Nucleotide Resolution</a>   Transformer + gene</p>
</li>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.1202529">Synaptic Pruning by Microglia Is Necessary for Normal Brain Development</a></p>
</li>
</ul>
<h3 id="cognition">cognition</h3>
<h4 id="attention">attention</h4>
<h4 id="learning-and-memory">learning and memory</h4>
<ul>
<li><a href="https://journals.physiology.org/doi/full/10.1152/jn.00454.2022">How our understanding of memory replay evolves</a></li>
</ul>
<h4 id="decision-making">decision making</h4>
<ul>
<li>
<p><a href="https://www.jneurosci.org/content/28/12/3017">Bounded Integration in Parietal Cortex Underlies Decisions Even When Viewing Duration Is Dictated by the Environment</a></p>
</li>
<li>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(23)00758-4">Dopamine pathways mediating affective state transitions after sleep loss</a></p>
</li>
</ul>
<h3 id="ds--or">DS &amp; OR</h3>
<ul>
<li><a href="https://en.wikipedia.org/wiki/Association_rule_learning">Association rule learning</a></li>
</ul>
<h4 id="quadratic-assignment-problem">Quadratic assignment problem</h4>
<ul>
<li>
<p><a href="https://en.wikipedia.org/wiki/Quadratic_assignment_problem">quadratic assignment problem</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1905.10124">Sliced Gromov-Wasserstein</a></p>
</li>
<li>
<p><a href="https://pythonot.github.io/gen_modules/ot.gromov.html#module-ot.gromov">ot.gromov</a>
Fused Gromov-Wasserstein transport</p>
</li>
</ul>
<h4 id="similarity-matrix">similarity matrix</h4>
<ul>
<li>
<p><a href="https://arxiv.org/abs/1502.07541">Euclidean Distance Matrices: Essential Theory, Algorithms and Applications</a></p>
</li>
<li>
<p><a href="https://www.frontiersin.org/articles/10.3389/neuro.06.004.2008/full">Representational similarity analysis – connecting the branches of systems neuroscience</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1502.07541">Euclidean Distance Matrices: Essential Theory, Algorithms and Applications</a></p>
</li>
<li>
<p><strong>CCA</strong> <a href="https://www.worldscientific.com/doi/abs/10.1142/S012906570000034X">Kernel and nonlinear canonical correlation analysis</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1905.00414">Similarity of Neural Network Representations Revisited</a>
Simon Kornblith, Mohammad Norouzi, Honglak Lee, Geoffrey Hinton</p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2211.11665">Representational dissimilarity metric spaces for stochastic neural networks</a>
Lyndon R. Duong, Jingyang Zhou, Josue Nassar, Jules Berman, Jeroen Olieslagers, Alex H. Williams</p>
</li>
</ul>
<h3 id="ml-1">ML</h3>
<ul>
<li>
<p><a href="https://arxiv.org/abs/2305.14608">Inverse Reinforcement Learning with the Average Reward Criterion</a></p>
</li>
<li>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(20)30705-4">Artificial Neural Networks for Neuroscientists: A Primer</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1706.03762">Attention Is All You Need</a>
<strong>Transformer</strong></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1506.04214">Convolutional LSTM Network: A Machine Learning Approach for Precipitation Nowcasting</a>
recurrent CNN</p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2210.02747">Flow Matching for Generative Modeling</a> 2022</p>
</li>
<li></li>
</ul>
<h4 id="nas">NAS</h4>
<ul>
<li><a href="https://arxiv.org/abs/1902.09635">NAS-Bench-101: Towards Reproducible Neural Architecture Search</a>
Chris Ying, Aaron Klein, Esteban Real, Eric Christiansen, Kevin Murphy, Frank Hutter</li>
</ul>
<h4 id="cnn-kernel">CNN kernel</h4>
<p>Clustering convolutional kernels</p>
<ul>
<li>
<p><a href="https://www.worldscientific.com/doi/epdf/10.1142/S0219467819500190">Framework for Automatic Selection of Kernels based on Convolutional Neural Networks and CkMeans Clustering Algorithm</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1804.11191">How convolutional neural network see the world - A survey of convolutional neural network visualization methods</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1311.2901">Visualizing and Understanding Convolutional Networks</a></p>
</li>
</ul>
<h3 id="sparse-decoding">sparse decoding</h3>
<ul>
<li><a href="https://arxiv.org/abs/2110.10804">Identifiable Deep Generative Models via Sparse Decoding</a></li>
</ul>
<p>UFLDL Tutorial</p>
<ul>
<li>
<p><a href="http://ufldl.stanford.edu/tutorial/unsupervised/SparseCoding/">Sparse Coding</a></p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/abstract/document/5872076">Sensing Matrix Optimization for Block-Sparse Decoding</a></p>
</li>
<li>
<p><a href="https://ieeexplore.ieee.org/abstract/document/9782313">A Tutorial on Decoding Techniques of Sparse Code Multiple Access</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/0801.1241">On the iterative decoding of sparse quantum codes</a></p>
</li>
</ul>
<h3 id="others-1">others</h3>
<ul>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.01.27.577215v1">Mapping the developmental structure of stereotyped and individual-unique behavioral spaces in C. elegans</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/1910.01312">A sparse semismooth Newton based augmented Lagrangian method for large-scale support vector machines</a> SVM</p>
</li>
<li>
<p><a href="https://www.annualreviews.org/doi/abs/10.1146/annurev.neuro.31.060407.125555">Vestibular System: The Many Facets of a Multimodal Sense</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627321007054">Spatial modulation of hippocampal activity in freely moving macaques</a>
Dora E. Angelaki</p>
</li>
<li>
<p><a href="https://www.brains-explained.com/how-hubel-and-wiesel-revolutionized-neuroscience/">How Hubel and Wiesel Revolutionized Neuroscience and Made Me a Neuroscientist</a>
a story</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41583-023-00764-z">Sleep and circadian rhythmicity as entangled processes serving homeostasis</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627324000473">Tuned geometries of hippocampal representations meet the computational demands of social memory</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41467-023-37980-1">Complex computation from developmental priors</a></p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1053811919300114">A practical guide to linking brain-wide gene expression and neuroimaging data</a>
gene</p>
</li>
</ul>
<h3 id="wiki">wiki</h3>
<ul>
<li>
<p>wiki <a href="https://en.wikipedia.org/wiki/Non-negative_least_squares">Non-negative least squares</a></p>
</li>
<li>
<p>wiki <a href="https://en.wikipedia.org/wiki/Kronecker_product">Kronecker product</a></p>
</li>
<li>
<p>wiki <a href="https://en.wikipedia.org/wiki/F-score">F-score</a></p>
</li>
<li>
<p>wiki <a href="https://en.wikipedia.org/wiki/Precision_and_recall">Precision_and_recall</a></p>
</li>
</ul>
<h1 id="todo">TODO</h1>
<ul>
<li>
<p><a href="https://www.nature.com/articles/s41586-022-05562-8">Synaptic gradients transform object location to action</a>
S. Lawrence Zipursky &amp; Gwyneth M. Card</p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41467-024-50608-2">Whole brain alignment of spatial transcriptomics between humans and mice with BrainAlign</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.09.11.557287v3">Cell2Sentence: Teaching Large Language Models the Language of Biology</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/nmeth.4402">Reversed graph embedding resolves complex single-cell trajectories</a></p>
</li>
<li>
<p><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4772-0">Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics</a></p>
</li>
<li>
<p><a href="https://dl.acm.org/doi/pdf/10.1145/1467247.1467259#page=1.25">viewpoint Your Students are Your legacy</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2025.09.30.679510v1.full.pdf">The central complex of the larval fruit fly brain</a></p>
</li>
<li>
<p><a href="https://arxiv.org/abs/2501.00278">A data-driven biophysical network model reproduces C. elegans premotor neural dynamics</a></p>
</li>
<li>
<p><a href="https://arxiv.org/pdf/2404.07783">Synchronization of two non-identical Chialvo neurons</a></p>
</li>
<li>
<p><a href="https://siplab.gatech.edu/people.html">Christopher Rozell</a></p>
</li>
<li>
<p>Quantitative understanding of brain activity, the connection between the brain and the body, and the effects of neural stimulation are grand challenges with the potential for enormous societal impact. Simultaneous advances in neural interfacing and data science have created a remarkable opportunity to reshape the way we think about the brain in health and disease</p>
</li>
<li>
<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0925231221002393">Outlier exposure with confidence control for out-of-distribution detection</a></p>
</li>
<li>
<p><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC6093779/">Drosophila as a model system for neurotransmitter measurements</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41586-024-08392-y">Episodic and associative memory from spatial scaffolds in the hippocampus</a></p>
</li>
<li>
<p><a href="https://www.science.org/doi/10.1126/science.1194869">Universality in the Evolution of Orientation Columns in the Visual Cortex</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.06.20.599966v1">Mapping Individual Differences in the Topological Landscape of Naturalistic Brain Dynamics</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/35002125">A synthetic oscillatory network of transcriptional regulators</a></p>
</li>
<li>
<p><a href="https://www.nature.com/articles/s41583-025-00954-x">The emergence of NeuroAI: bridging neuroscience and artificial intelligence</a></p>
</li>
<li>
<p><a href="https://www.science.org/doi/abs/10.1126/science.ads7633">Dimensionality reduction simplifies synaptic partner matching in an olfactory circuit</a></p>
</li>
<li>
<p><a href="https://link.springer.com/article/10.1007/s12038-025-00518-2">Enhanced whole-brain calcium imaging and cell identification in C. elegans reveal AWCOFF neuronal responses to 2-nonanone</a></p>
</li>
<li>
<p><a href="https://www.biorxiv.org/content/biorxiv/early/2025/05/21/2025.05.16.654614.1.full.pdf">Aligning Neural Population Patterns Facilitates Motor Learning Transfer</a></p>
</li>
<li>
<p><a href="https://arxiv.org/pdf/2006.11239">Denoising Diffusion Probabilistic Models</a></p>
</li>
<li>
<p><a href="https://arxiv.org/pdf/2508.05926">Reverse Diffusion Sequential Monte Carlo Samplers</a></p>
</li>
<li>
<p><a href="https://arxiv.org/pdf/2508.00191">STATE-SWITCHING NAVIGATION STRATEGIES IN C. elegans ARE BENEFICIAL FOR CHEMOTAXIS</a></p>
</li>
</ul>
<p><a href="https://www.nature.com/articles/s41592-025-02687-2">NeuroXiv: AI-powered open databasing and dynamic mining of brain-wide neuron morphometry</a></p>

            <script async src="https://cdn.jsdelivr.net/npm/katex-copytex@latest/dist/katex-copytex.min.js"></script>
            
        </body>
        </html>