<!DOCTYPE html>
        <html>
        <head>
            <meta charset="UTF-8">
            <title>paper reading list</title>
            <style>
/* From extension vscode.github */
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

.vscode-dark img[src$=\#gh-light-mode-only],
.vscode-light img[src$=\#gh-dark-mode-only] {
	display: none;
}

</style>
            
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">
<style>
            body {
                font-family: -apple-system, BlinkMacSystemFont, 'Segoe WPC', 'Segoe UI', system-ui, 'Ubuntu', 'Droid Sans', sans-serif;
                font-size: 14px;
                line-height: 1.6;
            }
        </style>
        <style>
.task-list-item {
    list-style-type: none;
}

.task-list-item-checkbox {
    margin-left: -20px;
    vertical-align: middle;
    pointer-events: none;
}
</style>
<style>
:root {
  --color-note: #0969da;
  --color-tip: #1a7f37;
  --color-warning: #9a6700;
  --color-severe: #bc4c00;
  --color-caution: #d1242f;
  --color-important: #8250df;
}

</style>
<style>
@media (prefers-color-scheme: dark) {
  :root {
    --color-note: #2f81f7;
    --color-tip: #3fb950;
    --color-warning: #d29922;
    --color-severe: #db6d28;
    --color-caution: #f85149;
    --color-important: #a371f7;
  }
}

</style>
<style>
.markdown-alert {
  padding: 0.5rem 1rem;
  margin-bottom: 16px;
  color: inherit;
  border-left: .25em solid #888;
}

.markdown-alert>:first-child {
  margin-top: 0
}

.markdown-alert>:last-child {
  margin-bottom: 0
}

.markdown-alert .markdown-alert-title {
  display: flex;
  font-weight: 500;
  align-items: center;
  line-height: 1
}

.markdown-alert .markdown-alert-title .octicon {
  margin-right: 0.5rem;
  display: inline-block;
  overflow: visible !important;
  vertical-align: text-bottom;
  fill: currentColor;
}

.markdown-alert.markdown-alert-note {
  border-left-color: var(--color-note);
}

.markdown-alert.markdown-alert-note .markdown-alert-title {
  color: var(--color-note);
}

.markdown-alert.markdown-alert-important {
  border-left-color: var(--color-important);
}

.markdown-alert.markdown-alert-important .markdown-alert-title {
  color: var(--color-important);
}

.markdown-alert.markdown-alert-warning {
  border-left-color: var(--color-warning);
}

.markdown-alert.markdown-alert-warning .markdown-alert-title {
  color: var(--color-warning);
}

.markdown-alert.markdown-alert-tip {
  border-left-color: var(--color-tip);
}

.markdown-alert.markdown-alert-tip .markdown-alert-title {
  color: var(--color-tip);
}

.markdown-alert.markdown-alert-caution {
  border-left-color: var(--color-caution);
}

.markdown-alert.markdown-alert-caution .markdown-alert-title {
  color: var(--color-caution);
}

</style>
        
        </head>
        <body class="vscode-body vscode-light">
            <h1 id="paper-reading-list">paper reading list</h1>
<h2 id="lab-and-people">lab and people</h2>
<p><a href="https://www.internationalbrainlab.com/our-team">IBL</a></p>
<p><a href="https://scholar.google.com/citations?user=FayXlTYAAAAJ&amp;hl=en">Liam Paninski</a></p>
<p><a href="https://scholar.google.com/citations?user=-ElvJ9wAAAAJ&amp;hl=en">Jonathan Pillow</a></p>
<h2 id="connectome">Connectome</h2>
<p><a href="https://arxiv.org/abs/1912.08986">Deep Connectomics Networks: Neural Network Architectures Inspired by Neuronal Networks</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.06.08.544257v1">Not everything, not everywhere, not all at once: a study of brain-wide encoding of movement</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.05.30.542725v1">Synaptic architecture of leg and wing motor control networks in Drosophila</a></p>
<p><a href="https://doi.org/10.1162/netn_a_00044">NeuroCave: A web-based immersive visualization platform for exploring connectome datasets</a></p>
<p><a href="https://elifesciences.org/articles/62362">Accelerating with FlyBrainLab the discovery of the functional logic of the Drosophila brain in the connectomic and synaptomic era</a></p>
<p><a href="https://elifesciences.org/reviewed-preprints/91532#tab-content">Deciphering the Genetic Code of Neuronal Type Connectivity: A Bilinear Modeling Approach</a>
bilinear + Mu Qiao</p>
<p><a href="https://www.nature.com/articles/s41583-023-00756-z">How deep is the brain? The shallow brain hypothesis</a></p>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(23)00719-5">Mapping of neuronal and glial primary cilia contactome and connectome in the human cerebral cortex</a></p>
<h3 id="technology-of-connectome">technology of connectome</h3>
<p><a href="https://openreview.net/forum?id=lHDVYDy5S9l">Bridging the Gap: Point Clouds for Merging Neurons in Connectomics</a>
MIDL 2022</p>
<h3 id="connectome-analysis">connectome analysis</h3>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.06.27.546055v2">Whole-brain annotation and multi-connectome cell typing quantifies circuit stereotypy in Drosophila</a>
<strong>hemibrain and flywire</strong></p>
<p><a href="https://elifesciences.org/articles/57443">A connectome and analysis of the adult Drosophila central brain</a><br>
<strong>hemibrain</strong></p>
<p><a href="https://elifesciences.org/articles/79139">Connectomic analysis of the Drosophila lateral neuron clock cells reveals the synaptic basis of functional pacemaker classes</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1053811920308326">Cross-species functional alignment reveals evolutionary hierarchy within the connectome</a></p>
<h3 id="connectome-and-gene">connectome and gene</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867421007583">Molecular topography of an entire nervous system</a>
<strong>key_and_lock</strong></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0960982223010631">Brain wiring determinants uncovered by integrating connectomes and transcriptomes</a>
<strong>Yoo2023</strong> T4_T5</p>
<p><strong>bilinear</strong> <a href="https://www.pnas.org/doi/abs/10.1073/pnas.2009093117">Uncovering the genetic blueprint of the C. elegans nervous system</a>
István A. Kovács, Dániel L. <strong>Barabási</strong>, and Albert-László <strong>Barabási</strong>  PNAS</p>
<p><strong>biclique motifs</strong> <a href="https://www.sciencedirect.com/science/article/pii/S0896627319309262">A Genetic Model of the Connectome</a>
Dániel L. Barabási 1, Albert-László Barabási</p>
<p><strong>bilinear</strong> <a href="https://elifesciences.org/reviewed-preprints/91532">Deciphering the Genetic Code of Neuronal Type Connectivity: A Bilinear Modeling Approach</a>
<strong>Mu Qiao</strong></p>
<h3 id="gene-data">gene data</h3>
<p><a href="https://www.nature.com/articles/s41586-021-04262-z">Decoding gene regulation in the fly brain</a></p>
<p><a href="https://www.nature.com/articles/s41586-021-04262-z">Decoding gene regulation in the fly brain</a>
<a href="https://www.nature.com/articles/s41586-022-04564-w">A complete temporal transcription factor series in the fly visual system</a>
Claude Desplan</p>
<h4 id="application">application</h4>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.03.16.435261v2">Encoding innate ability through a genomic bottleneck</a></p>
<h3 id="connectome-visualization">connectome visualization</h3>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.07.26.550598v1">CAVE: Connectome Annotation Versioning Engine</a>
<a href="https://vcg.seas.harvard.edu/publications/cave">https://vcg.seas.harvard.edu/publications/cave</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.10.05.561103v1">SmartEM: machine-learning guided electron microscopy</a></p>
<p>BOOK: <a href="https://link.springer.com/chapter/10.1007/978-1-4471-6497-5_21">Visualization in Connectomics</a></p>
<h2 id="neuroai">NeuroAI</h2>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(22)00806-6#%20">In vitro neurons learn and exhibit sentience when embodied in a simulated game-world</a>
<a href="https://www.nature.com/articles/s41928-023-01069-w">Brain organoid reservoir computing for artificial intelligence</a></p>
<h3 id="connectome--ann">Connectome &amp; ANN</h3>
<p><a href="connectome%20inspired%20neural%20network.html">connectome inspired neural network</a><br>
html <a href="https://lengyuner.github.io/blog/connectome%20inspired%20neural%20network.html">connectome inspired neural network</a></p>
<h3 id="geometry">geometry</h3>
<p><a href="https://openreview.net/forum?id=dqWobzlAGb">Modelling brain connectomes networks: Solv is a worthy competitor to hyperbolic geometry!</a>
ICLR 2024 Conference Submission</p>
<h3 id="brain-simulation--activity-prediction">Brain simulation / activity prediction</h3>
<p><a href="https://openreview.net/forum?id=7ntI4kcoqG">AMAG: Additive, Multiplicative and Adaptive Graph Neural Network For Forecasting Neuron Activity</a>
NeurIPS 2023 poster</p>
<h4 id="model">model</h4>
<p><a href="https://arxiv.org/abs/2002.09405">Learning to Simulate Complex Physics with Graph Networks</a>
为了做这个，我们需要什么样的data。有wide field and the 2p microscope，是不是就可以模拟一些基础规律了</p>
<h3 id="network-analysis">Network analysis</h3>
<p>BOOK <a href="https://www.sciencedirect.com/book/9780124079083/fundamentals-of-brain-network-analysis">Fundamentals of Brain Network Analysis</a></p>
<p>BOOK <a href="https://www.sciencedirect.com/science/article/pii/B9780124079083000030">Chapter 3 - Connectivity Matrices and Brain Graphs</a></p>
<p>Review <a href="https://www.frontiersin.org/articles/10.3389/fnsys.2020.609316/full">Neural Networks With Motivation</a></p>
<h3 id="synapse-plasticity">synapse plasticity</h3>
<h4 id="stdp">STDP</h4>
<p>nearest neighbor STDP (Izhikevich, 2003)
<a href="https://doi.org/10.1162/089976603321891783">Relating STDP to BCM</a></p>
<p><a href="https://arxiv.org/abs/2310.20309">Tensor formalism for predicting synaptic connections with ensemble modeling or optimization</a>
Tirthabir Biswas, Tianzhi Lambus Li, James E. Fitzgerald</p>
<h2 id="neural-data-analysis">neural data analysis</h2>
<p><a href="https://direct.mit.edu/neco/article/27/1/104/8035/Spatiotemporal-Conditional-Inference-and">Spatiotemporal conditional inference and hypothesis tests for neural ensemble spiking precision</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0959438816302689">Maximum entropy models as a tool for building precise neural controls</a>
Cristina Savin, Gašper Tkačik</p>
<h3 id="reivew">reivew</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0959438817302428">Neural data science: accelerating the experiment-analysis-theory cycle in large-scale neuroscience </a>
ICCV, 2019</p>
<h3 id="high-dimensional-data">high dimensional data</h3>
<p><a href="https://www.science.org/doi/10.1126/science.aav7893">Spontaneous behaviors drive multidimensional, brainwide activity</a>
Carsen Stringer</p>
<p><a href="https://www.nature.com/articles/s41586-019-1346-5">High-dimensional geometry of population responses in visual cortex</a>
Carsen Stringer</p>
<p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0896627324001211">Simultaneous, cortex-wide dynamics of up to <strong>1 million neurons</strong> reveal unbounded scaling of dimensionality with neuron number</a></p>
<h3 id="dimensional-reduction">dimensional reduction</h3>
<p>Phate: <a href="https://www.nature.com/articles/s41587-019-0336-3">Visualizing structure and transitions in high-dimensional biological data</a>
Phate</p>
<h2 id="encoding-and-decoding">encoding and decoding</h2>
<p><a href="https://www.science.org/doi/10.1126/science.1234330">Neural Decoding of Visual Imagery During Sleep</a></p>
<p><a href="https://www.jneurosci.org/content/37/14/3764.abstract">Locomotion Enhances Neural Encoding of Visual Stimuli in Mouse V1</a></p>
<p><a href="https://journals.physiology.org/doi/full/10.1152/jn.01046.2012">Bayesian decoding using unsorted spikes in the rat hippocampus</a>
Fabian Kloosterman, Stuart P. Layton, Zhe Chen, and Matthew A. Wilson</p>
<h3 id="deep-learning-method">deep learning method</h3>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.09.18.558113v1">Neural Data Transformer 2: Multi-context Pretraining for Neural Spiking Activity</a></p>
<p>LFADS: <a href="https://www.nature.com/articles/s41592-018-0109-9">Inferring single-trial neural population dynamics using sequential auto-encoders</a><br>
<a href="https://arxiv.org/abs/1608.06315">LFADS (method part)</a></p>
<h4 id="">?</h4>
<p><a href="https://openreview.net/forum?id=90O5cvFZkZ">GUST: Combinatorial Generalization by Unsupervised Grouping with Neuronal Coherence</a>
NeurIPS 2023 poster</p>
<h2 id="behavior">behavior</h2>
<p>GLM-HMM: <a href="https://www.nature.com/articles/s41593-019-0533-x">Unsupervised identification of the internal states that shape natural behavior</a><br>
<a href="https://www.nature.com/articles/s41593-021-01007-z">Mice alternate between discrete strategies during perceptual decision-making</a>
Jonathan W. Pillow</p>
<p><a href="https://arxiv.org/abs/2007.08723">End-to-end Deep Prototype and Exemplar Models for Predicting Human Behavior</a>
Thomas L. Griffiths</p>
<p><a href="https://www.frontiersin.org/articles/10.3389/fncir.2012.00108/full">Spatial vision in insects is facilitated by shaping the dynamics of visual input through behavioral action</a></p>
<p><a href="https://www.nature.com/articles/nature14855">Arithmetic and local circuitry underlying dopamine prediction errors</a></p>
<p><a href="https://www.science.org/doi/10.1126/science.275.5306.1593">A Neural Substrate of Prediction and Reward</a></p>
<p><a href="https://openreview.net/forum?id=nosngu5XwY9">Dynamic Inverse Reinforcement Learning for Characterizing Animal Behavior</a>  Zoe Ashwood, Aditi Jha, Jonathan W. Pillow<br>
NeurIPS 2022</p>
<p><a href="https://www.nature.com/articles/s41586-022-05611-2">Spontaneous behaviour is structured by reinforcement without explicit reward</a>
Sandeep Robert Datta</p>
<h3 id="oscilation">oscilation</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627323002143">Brain rhythms have come of age</a></p>
<h2 id="vision">vision</h2>
<p><a href="https://www.pnas.org/doi/full/10.1073/pnas.1922294117">The limits of color awareness during active, real-world vision</a>
using VR to test the color fade perception</p>
<p><a href="https://www.frontiersin.org/articles/10.3389/fncir.2015.00033/full">A common evolutionary origin for the ON- and OFF-edge motion detection pathways of the Drosophila visual system</a>
Ian A. Meinertzhagen</p>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.12.20.473513v1">Neuronal circuits integrating visual motion information in Drosophila</a>
T4 and T5 neuron</p>
<h2 id="others">others</h2>
<h3 id="gene">gene</h3>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867416300034">Bayesian Sparse Regression Analysis Documents the Diversity of Spinal Inhibitory Interneurons</a></p>
<p><a href="https://arxiv.org/abs/2306.15794">HyenaDNA: Long-Range Genomic Sequence Modeling at Single Nucleotide Resolution</a>   Transformer + gene</p>
<p><a href="https://www.science.org/doi/10.1126/science.1202529">Synaptic Pruning by Microglia Is Necessary for Normal Brain Development</a></p>
<h3 id="morphology">morphology</h3>
<p><a href="https://journals.physiology.org/doi/full/10.1152/jn.00865.2006">Mapping Function Onto Neuronal Morphology</a>
Klaus M. Stiefel1, andTerrence J. Sejnowski</p>
<p><a href="https://www.cell.com/fulltext/S0896-6273(04)00498-2">Synaptic Connectivity and Neuronal Morphology: Two Sides of the Same Coin</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0165027010006758">Automated analysis of neuronal morphology, synapse number and synaptic recruitment</a></p>
<p><a href="https://www.nature.com/articles/nprot.2008.51">L-Measure: a web-accessible tool for the analysis, comparison and search of digital reconstructions of neuronal morphologies</a></p>
<p><a href="https://openreview.net/forum?id=p6hArCtwLAU">TreeMoCo: Contrastive Neuron Morphology Representation Learning</a></p>
<h4 id="maximum-weighted-independent-set-mwis">maximum weighted independent set (MWIS)</h4>
<p><a href="https://arxiv.org/abs/1810.10834">Exactly Solving the Maximum Weight Independent Set Problem on Large Real-World Graphs</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0166218X02002056">A note on greedy algorithms for the maximum weighted independent set problem</a></p>
<p>Graph compression or sparsification
<a href="https://arxiv.org/pdf/2007.07862.pdf">Vertex Sparsification for Edge Connectivity</a></p>
<h4 id="maximum-dispersion">Maximum Dispersion</h4>
<p>Maximum Diversity Problem (MDP)
<a href="https://arxiv.org/abs/1706.09185">Dispersion on Trees</a>
<a href="https://ieeexplore.ieee.org/document/7004250">Estimating Pairwise Distances in Large Graphs</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0196677400911453">Approximation Algorithms for Dispersion Problems</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0167637797000345">Approximation Algorithms for Maximum Dispersion</a>
<a href="https://epubs.siam.org/doi/10.1137/0404048">Obnoxious Facility Location on Graphs</a></p>
<p><a href="https://zhuanlan.zhihu.com/p/101724381">变邻域搜索(VNS)算法求解Max-Mean Dispersion Problem</a></p>
<p><a href="https://arxiv.org/abs/1503.00980">On memetic search for the max-mean dispersion problem</a>
<a href="https://www.sciencedirect.com/science/article/pii/S0950705115001860">Tabu search for the Max–Mean Dispersion Problem</a></p>
<!-- [Exact Solution of P-dispersion Problems](https://www.semanticscholar.org/paper/Exact-Solution-of-P-dispersion-Problems-Pisinger/1eb3810077c0af9d46ed5ff2b0819d954c97dcae) -->
<p><a href="https://www.sciencedirect.com/science/article/pii/S0305054804002552">Upper bounds and exact algorithms for p-dispersion problems
</a>
<a href="http://yalma.fime.uanl.mx/~roger/work/teaching/class_tso/docs_project/problems/PDP/cor-2006-Pisinger.pdf">http://yalma.fime.uanl.mx/~roger/work/teaching/class_tso/docs_project/problems/PDP/cor-2006-Pisinger.pdf</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0377221721006548">A review on discrete diversity and dispersion maximization from an OR perspective</a></p>
<p><a href="https://pubsonline.informs.org/doi/abs/10.1287/opre.42.2.299">Heuristic and special case algorithms for dispersion problems</a></p>
<p><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/net.3230120206">A network flow solution to some nonlinear 0-1 programming problems, with applications to graph theory</a></p>
<h3 id="place-cell-and-grid-cell">place cell and grid cell</h3>
<p><a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/medial-entorhinal-cortex">medial entorhinal cortex</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S009286742031388X">The Tolman-Eichenbaum Machine: Unifying Space and Relational Memory through Generalization in the Hippocampal Formation</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.10.28.466284v5">Emergence of robust global modules from local interactions and smooth gradients</a></p>
<h3 id="cognition">cognition</h3>
<h4 id="attention">attention</h4>
<h4 id="learning-and-memory">learning and memory</h4>
<p><a href="https://journals.physiology.org/doi/full/10.1152/jn.00454.2022">How our understanding of memory replay evolves</a></p>
<h4 id="decision-making">decision making</h4>
<p><a href="https://www.jneurosci.org/content/28/12/3017">Bounded Integration in Parietal Cortex Underlies Decisions Even When Viewing Duration Is Dictated by the Environment</a></p>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(23)00758-4">Dopamine pathways mediating affective state transitions after sleep loss</a></p>
<h3 id="ds">DS</h3>
<h4 id="similarity-matrix">similarity matrix</h4>
<p><a href="https://arxiv.org/abs/1502.07541">Euclidean Distance Matrices: Essential Theory, Algorithms and Applications</a></p>
<p><a href="https://www.frontiersin.org/articles/10.3389/neuro.06.004.2008/full">Representational similarity analysis – connecting the branches of systems neuroscience</a></p>
<p><a href="https://arxiv.org/abs/1502.07541">Euclidean Distance Matrices: Essential Theory, Algorithms and Applications</a></p>
<p><strong>CCA</strong> <a href="https://www.worldscientific.com/doi/abs/10.1142/S012906570000034X">Kernel and nonlinear canonical correlation analysis</a></p>
<p><a href="https://arxiv.org/abs/1905.00414">Similarity of Neural Network Representations Revisited</a>
Simon Kornblith, Mohammad Norouzi, Honglak Lee, Geoffrey Hinton</p>
<p><a href="https://arxiv.org/abs/2211.11665">Representational dissimilarity metric spaces for stochastic neural networks</a>
Lyndon R. Duong, Jingyang Zhou, Josue Nassar, Jules Berman, Jeroen Olieslagers, Alex H. Williams</p>
<h3 id="ml">ML</h3>
<p><a href="https://arxiv.org/abs/2305.14608">Inverse Reinforcement Learning with the Average Reward Criterion</a></p>
<p><a href="https://www.cell.com/neuron/fulltext/S0896-6273(20)30705-4">Artificial Neural Networks for Neuroscientists: A Primer</a></p>
<p><a href="https://arxiv.org/abs/1706.03762">Attention Is All You Need</a>
<strong>Transformer</strong></p>
<h4 id="nas">NAS</h4>
<p><a href="https://arxiv.org/abs/1902.09635">NAS-Bench-101: Towards Reproducible Neural Architecture Search</a>
Chris Ying, Aaron Klein, Esteban Real, Eric Christiansen, Kevin Murphy, Frank Hutter</p>
<h4 id="cnn-kernel">CNN kernel</h4>
<p>Clustering convolutional kernels</p>
<p><a href="https://www.worldscientific.com/doi/epdf/10.1142/S0219467819500190">Framework for Automatic Selection of Kernels based on Convolutional Neural Networks and CkMeans Clustering Algorithm</a></p>
<p><a href="https://arxiv.org/abs/1804.11191">How convolutional neural network see the world - A survey of convolutional neural network visualization methods</a></p>
<p><a href="https://arxiv.org/abs/1311.2901">Visualizing and Understanding Convolutional Networks</a></p>
<h3 id="others-1">others</h3>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.01.27.577215v1">Mapping the developmental structure of stereotyped and individual-unique behavioral spaces in C. elegans</a></p>
<p><a href="https://arxiv.org/abs/1910.01312">A sparse semismooth Newton based augmented Lagrangian method for large-scale support vector machines</a> SVM</p>
<p><a href="https://www.annualreviews.org/doi/abs/10.1146/annurev.neuro.31.060407.125555">Vestibular System: The Many Facets of a Multimodal Sense</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627321007054">Spatial modulation of hippocampal activity in freely moving macaques</a>
Dora E. Angelaki</p>
<p><a href="https://www.brains-explained.com/how-hubel-and-wiesel-revolutionized-neuroscience/">How Hubel and Wiesel Revolutionized Neuroscience and Made Me a Neuroscientist</a>
a story</p>
<p><a href="https://www.nature.com/articles/s41583-023-00764-z">Sleep and circadian rhythmicity as entangled processes serving homeostasis</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0896627324000473">Tuned geometries of hippocampal representations meet the computational demands of social memory</a></p>
<p><a href="https://www.nature.com/articles/s41467-023-37980-1">Complex computation from developmental priors</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1053811919300114">A practical guide to linking brain-wide gene expression and neuroimaging data</a>
gene</p>
<h3 id="wiki">wiki</h3>
<p>wiki <a href="https://en.wikipedia.org/wiki/Non-negative_least_squares">Non-negative least squares</a></p>
<p>wiki <a href="https://en.wikipedia.org/wiki/Kronecker_product">Kronecker product</a></p>
<p>wiki <a href="https://en.wikipedia.org/wiki/F-score">F-score</a></p>
<p>wiki <a href="https://en.wikipedia.org/wiki/Precision_and_recall">Precision_and_recall</a></p>

            
            
        </body>
        </html>